<html>
<link rel="stylesheet" type="text/css"
    href="https://2020.igem.org/Template:BIT/css/multi_drop_menu?action=raw&ctype=text/css" />

<script>
    function changeColortoGreen(e) {
        e.style.backgroundColor = 'rgb(31, 160, 174)';
    }

    function changeColortoBlack(e) {
        e.style.backgroundColor = 'rgb(51, 51, 51)';
    }
</script>

<!--这是返回顶部控件的css,不要修改，单独抽离后会失效-->
<style>
    .cd-top {
        display: inline-block;
        height: 40px;
        width: 40px;
        position: fixed;
        bottom: 40px;
        right: 10px;
        box-shadow: 0 0 10px rgba(0, 0, 0, 0.05);
        /* image replacement properties */
        overflow: hidden;
        text-indent: 100%;
        white-space: nowrap;
        background: rgba(232, 98, 86, 0.8) url(https://2020.igem.org/wiki/images/1/12/T--BIT--cd-top-arrow.svg) no-repeat center 50%;
        visibility: hidden;
        opacity: 0;
        -webkit-transition: opacity .3s 0s, visibility 0s .3s;
        -moz-transition: opacity .3s 0s, visibility 0s .3s;
        transition: opacity .3s 0s, visibility 0s .3s;
    }

    .cd-top.cd-is-visible,
    .cd-top.cd-fade-out,
    .no-touch .cd-top:hover {
        -webkit-transition: opacity .3s 0s, visibility 0s 0s;
        -moz-transition: opacity .3s 0s, visibility 0s 0s;
        transition: opacity .3s 0s, visibility 0s 0s;
    }

    .cd-top.cd-is-visible {
        /* the button becomes visible */
        visibility: visible;
        opacity: 1;
    }

    .cd-top.cd-fade-out {
        /* if the user keeps scrolling down, the button is out of focus and becomes less visible */
        opacity: .5;
    }

    .no-touch .cd-top:hover {
        background-color: #e86256;
        opacity: 1;
    }

    @media only screen and (min-width: 768px) {
        .cd-top {
            right: 20px;
            bottom: 20px;
        }
    }

    @media only screen and (min-width: 1024px) {
        .cd-top {
            height: 60px;
            width: 60px;
            right: 30px;
            bottom: 30px;
        }
    }
</style>

<body>
    <div id="main-content">
        <header>
            <div
                style="background: rgb(34, 34, 34); position: fixed; width: 81%; box-shadow: 0 0 10px rgba(0,0, 0, .5)">
                <br>
                <div id="menu_box" style="float: right">
                    <div id="multi_drop_menu">
                        <!--一级菜单开始-->
                        <ul>
                            <li><a href="">Home</a>
                                <ul>
                                    <li><a href="https://2020.igem.org/Team:BIT">Home</a></li>
                                </ul>
                            </li>
                            <li><a href="">Project</a>
                                <!--二级菜单开始-->
                                <ul>
                                    <li><a href="https://2020.igem.org/Team:BIT/Description">Desciption</a>
                                    <li><a href="https://2020.igem.org/Team:BIT/Bio">Bio</a></li>
                                    <li><a href="https://2020.igem.org/Team:BIT/Hardware">Hardware</a></li>
                                    <li><a href="https://2020.igem.org/Team:BIT/Model">Model</a></li>
                                    <li><a href="https://2020.igem.org/Team:BIT/Engineering">Engineering</a></li>
                                    <li><a href="https://2020.igem.org/Team:BIT/Implementation">Implementation</a></li>
                                    <li><a href="https://2020.igem.org/Team:BIT/Proof_Of_Concept">Proof Of Concept
                                            &nbsp;</a></li>
                                    <li><a href="https://2020.igem.org/Team:BIT/Contribution">Contribution</a></li>
                                </ul>
                            </li>
                            <li><a href="">Practice</a>
                                <!--二级菜单开始-->
                                <ul>
                                    <li><a href="https://2020.igem.org/Team:BIT/Human_Practices">Overview</a></li>
                                    <li><a href="https://2020.igem.org/Team:BIT/Silver">Silver</a></li>
                                    <li><a href="https://2020.igem.org/Team:BIT/Integrated_Practice">Integrated Practice
                                            &nbsp;</a></li>
                                    <li><a href="https://2020.igem.org/Team:BIT/Education">Education</a></li>
                                </ul>
                            </li>
                            <li><a href="">Judge</a>
                                <!--二级菜单开始-->
                                <ul>
                                    <li><a href="https://2020.igem.org/Team:BIT/Medal">Medal</a></li>
                                    <li><a href="https://2020.igem.org/Team:BIT/Safety">Safety</a></li>
                                </ul>
                            </li>
                            <li><a href="">Team</a>
                                <!--二级菜单开始-->
                                <ul>
                                    <li><a href="https://2020.igem.org/Team:BIT/Members">Members</a></li>
                                    <li><a href="https://2020.igem.org/Team:BIT/Collaborations">Collaborations</a></li>
                                    <li><a href="https://2020.igem.org/Team:BIT/Partnership">Partnership</a></li>
                                    <li><a href="https://2020.igem.org/Team:BIT/Notebook">Notebook</a></li>
                                    <li><a href="https://2020.igem.org/Team:BIT/Attributions">Attributions</a></li>
                                    <li><a href="https://2020.igem.org/Team:BIT/Excellence_in_Another_Area">Excellence
                                            in Another Area &nbsp;</a></li>
                                </ul>
                            </li>
                        </ul>
                    </div>
                </div>

                <br>
                <br>
                <br>
            </div>
            <br>
            <br>
        </header>

        <br>
        <br>
        <br>
        <nav>
            <!--这里的&nbsp是占位用的，修改后整个页面会乱掉-->
            <div class="hidden">&nbsp;</div>

            <div class="inner">
                <div class="contact">
                    <p style="color: rgb(255, 255, 255);">CONNECT WITH US</p>
                    <hr>
                    <br>

                    <!-- <img src="https://2020.igem.org/wiki/images/9/97/T--BIT--youxiang.svg" style="background-color: rgb(51, 51, 51); width: 3rem;">
                <br> -->
                    <span style="color: rgb(255, 255, 255);">email: igem_bit@outlook.com </span><br>

                    <br>


                    <a href="https://weibo.com/u/5615996310" style="text-decoration: none;">
                        <img onmouseover="changeColortoGreen(this)" onmouseout="changeColortoBlack(this)"
                            src="https://2020.igem.org/wiki/images/9/90/T--BIT--weibo.svg"
                            style="background-color: rgb(51, 51, 51); width: 3rem;">
                    </a>



                    <a class="a-img" href="https://mobile.twitter.com/iGEM_BIT" style="text-decoration: none">
                        <img onmouseover="changeColortoGreen(this)" onmouseout="changeColortoBlack(this)"
                            src="https://2020.igem.org/wiki/images/2/21/T--BIT--ttww.svg"
                            style="background-color: rgb(51, 51, 51); width: 3rem;">
                    </a>



                    <a href="https://www.instagram.com/igem_bit" style="text-decoration: none">
                        <img onmouseover="changeColortoGreen(this)" onmouseout="changeColortoBlack(this)"
                            src="https://2020.igem.org/wiki/images/1/17/T--BIT--instagram.svg"
                            style="background-color: rgb(51, 51, 51); width: 3rem;">
                    </a>


                    <a href="https://github.com/a-layfolk/igem" style="text-decoration: none">
                        <img onmouseover="changeColortoGreen(this)" onmouseout="changeColortoBlack(this)"
                            src="https://2020.igem.org/wiki/images/1/10/T--BIT--git.svg"
                            style="background-color: rgb(51, 51, 51); width: 3rem;">
                    </a>



                    <br>

                </div>

                <div class="nav-bar">
                    <p style="color: rgb(255, 255, 255)">BOOKMARK</p>
                    <hr>
                    <br>
                    <ul>
                        <!-- 无序列表 -->
                        <li><a href="#1" style="color: rgb(255, 255, 255)">2.2.1 Background</a></li>
                        <li><a href="#2" style="color: rgb(255, 255, 255)">2.2.2 steps and principles</a>
                            <ul>
                                <li><a href="#2221" style="color: rgb(255, 255, 255)">2.2.2.1 sample preparation</a>
                                </li>
                                <li><a href="#2222" style="color: rgb(255, 255, 255)">2.2.2.2 aptamer replacement</a>
                                </li>
                                <li><a href="#2223" style="color: rgb(255, 255, 255)">2.2.2.3 isothermal amplification
                                        of RCA and HCR</a></li>
                                <li><a href="#2224" style="color: rgb(255, 255, 255)">2.2.2.4 nucleic acid signal
                                        detection</a></li>
                            </ul>
                        </li>
                        <li>
                            <a href="#3" style="color: rgb(255, 255, 255)">2.2.3 analysis of innovation points</a>
                        </li>
                        <li>
                            <a href="#4" style="color:rgb(255, 255, 255)">3. References</a>
                        </li>
                    </ul>
                    <br>
                </div>
            </div>
        </nav>



        <article>
            <a name="1"></a>
            <div
                style="background: url(https://2020.igem.org/wiki/images/e/eb/T--BIT--autumn1.jpg) no-repeat; height: 400px; background-size: 100%;">
            </div>

            <div class="inner">
                <br>
                <h1 style="color:white ; margin-top: -20%">BIO</h1>
                <br>
                <br>
                <br>
                <br>
                <br>
                <br>
                <br>
                <br>
                <br>
                <h2 style="color: rgb(237, 88, 58)"> 2.2.1 Background </h2>
                <br>

                <p>
                    &nbsp;
                    The biosensor module aims to build a fast, stable, highly accurate and highly sensitive detection
                    method and eventually achieve the real-time detection of the new coronavirus. At the same time, the
                    biological module is responsible for the conversion of biological signals into optical signals.
                    <br>
                    <br>
                    &nbsp;
                    Novel coronavirus nucleic acid and protein markers were jointly detected by the initial idea. The
                    nucleic acid signals were obtained by RPA isothermal amplification technology and then by
                    fluorescence technology and fluorescence strip detection with fluorescent probes based on the
                    SHERLOCK technology based on the principle of CRISPR-Cas. The specific working mechanism can be
                    summarized as follows: the samples of respiratory tract extract (deep expectoration liquid, nasal
                    swab, etc.) were pretreated to obtain nucleic acid signal, and the protein signal to be detected was
                    converted into nucleic acid signal through the designed aptamer system. The two kinds of signals
                    were amplified by RPA isothermal amplification method, and then the Ca in Sherlock technology was
                    used The Miss target defect effect of S protein makes the fluorescent probe display the fluorescence
                    signal for fluorescence detection, and provides the visual results through the lateral
                    chromatography strip technology (here we have considered two detection methods based on colloidal
                    gold and fluorescence immunity). This part mainly includes the construction of sample pretreatment,
                    the construction of multi nucleic acid signal isothermal amplification, and the construction of
                    nucleic acid signal detection and energy exchange.
                    <br>
                    <br>
                    &nbsp;
                    Subsequently, considering that the complementary chain obtained by aptamer replacement in protein
                    detection is very short, within 20 BP, which does not meet the requirements of RPA technology, we
                    adjusted the project in the middle of the project, and transformed RPA isothermal amplification
                    technology into HCR and RCA isothermal amplification technology, which is more suitable for the
                    detection of nucleic acid fragments of new coronavirus (here HCR technology is used for the
                    amplification of pure nucleic acid signal) RCA technology is used for amplification of nucleic acid
                    signal after the conversion of protein signal pathway by aptamer), but it retains the
                    characteristics of isothermal amplification, and still provides convenience for the temperature
                    control module of the hardware part of the project. At the same time, because Sherlock technology is
                    based on RPA technology, in line with the T7 transcription process in the technology, and because we
                    have no early accumulation of the system, and the technology has been very mature, we modify and
                    delete the part of the signal detection system based on CRISPR CAS principle. The working mechanism
                    of the interim project can be summarized as follows: the samples of respiratory tract extracts (deep
                    expectoration liquid, nasal swab, etc.) were pretreated and extracted to obtain the nucleic acid
                    signal to be detected, and the protein signal to be detected was converted into nucleic acid signal
                    through the designed aptamer system. The two signals were amplified by HCR and RCA isothermal
                    amplification technology, respectively, and were detected by fluorescence detection and fluorescence
                    detection The test results were displayed on the light immunoassay strip. This part mainly includes
                    the construction of sample pretreatment, the construction of two kinds of signal isothermal
                    amplification, and the construction of nucleic acid signal detection.
                    <br>
                    <br>
                    &nbsp;
                    Finally, considering that the accounting marker detection pathway has been widely studied in the
                    world, and due to time reasons, we don't think it is necessary to repeat the completed experiments.
                    Therefore, at the end of the project, we finally decided to abandon the nucleic acid marker
                    detection pathway, retain and highlight the innovation of the project team in the protein marker
                    detection pathway, but this does not affect the Let's combine the two markers to continue to be used
                    in the experiment. So far, the working mechanism at the end of the project group can be summarized
                    as follows: the respiratory tract extract samples (deep expectoration liquid, nasal swab, etc.) of
                    the subjects were pretreated and then extracted to obtain the protein signal to be tested (in the
                    experiment, we designed plasmids to express n protein for further separation and purification).
                    Then, the target complementary chain nucleic acid sequence was obtained by aptamer replacement
                    system, and then the obtained nucleic acid sequence was amplified by HCR and RCA isothermal
                    amplification methods. Finally, the detection results were displayed by fluorescence immunoassay
                    strip and colloidal gold strip. Our advantage is that novel coronavirus replacement system for
                    protein signal processing is very innovative. We amplified the signal by colloidal gold technique of
                    multiple isothermal amplification and HCR and RCA, which is more suitable for the detection of new
                    coronavirus and has the characteristics of mutual verification, sensitivity and accuracy. We finally
                    used colloidal gold technique strip, and quantitative display knots. The results make the data more
                    reliable. This part mainly includes the construction of sample pretreatment, aptamer replacement
                    system, isothermal amplification of nucleic acid signal, and construction of nucleic acid signal
                    detection.
                </p>
                <br>
                <div align="center">
                    <img src="https://2020.igem.org/wiki/images/8/81/T--BIT--BIO_1.png" style="box-shadow: 0px 0px 4px rgba(0, 0, 0, .5);
             width: 70%">
                    <div style="font-size: .8rem;">Figure 1.Experimental flow chart</div>
                </div>


                <a name="2"></a>
                <a name="2221"></a>
                <br>
                <br>
                <br>
                <br>
                <br>
                <h2 style="color: rgb(237, 88, 58)"> 2.2.2 steps and principles </h2>
                <br>
                <h3 style="color: rgb(237, 88, 58)"> 2.2.2.1 sample preparation </h3>
                <br>
                <h4 style="color: rgb(237, 88, 58)"> 1. Specimen collection </h4>
                <br>
                <p>
                    &nbsp;
                    For mild patients and suspected cases, most of the new coronavirus samples were deep cough sputum or
                    oropharyngeal swabs and nasopharynx swabs. The disposable virus sampling kit consists of sampling
                    swab, sampling tube, virus preservation solution and biosafety sample bag. The material of the
                    sampling head is closely related to the subsequent detection. Qualified sampling swabs should be
                    sterilized and packed, and the sampling swab head should be made of polyester (PE) synthetic fiber
                    or rayon (artificial fiber),
                    <br>
                    <br>
                    &nbsp;
                    When collecting virus samples from nasopharynx swabs, the following actions were taken: 1. The
                    sample holder held the swab with his right hand (a plastic rod swab with polypropylene fiber head),
                    and held the left hand on the top of the subject's head. 2. Make the swab with the top slightly bent
                    downward enter from the front nostril, and slowly go backward along the bottom of the lower nasal
                    meatus {as the nasal meatus is curved, the action should be gentle and slow, and do not use too much
                    force to avoid traumatic bleeding}. 3. When the top of the swab reaches the posterior wall of the
                    nasopharynx cavity (touching the wall), leave the swab for a moment (about 3 seconds), and then
                    rotate it gently for one week to slowly take out the swab. 4. Take another plastic rod swab with
                    polypropylene fiber head, collect the other side nostril in the same way, immerse the two swabs in
                    the same tube containing 3ml virus sample solution, discard the tail and screw the cap tightly.
                    <br>
                    <br>
                    &nbsp;
                    The saliva, oropharyngeal swab or nasopharynx swab were collected and stored in the virus sampling
                    tube. The commonly used virus preservation solution was divided into inactivated type and non
                    inactivated type. Inactivated virus preservative is transparent in color. It is characterized by
                    killing viruses and retaining nucleic acid fragments. It is commonly used to detect new coronavirus,
                    influenza virus, hand foot mouth disease and influenza A and so on.
                <div align="center">
                    <img src="https://2020.igem.org/wiki/images/a/a8/T--BIT--BIO_2.png" style="box-shadow: 0px 0px 4px rgba(0, 0, 0, .5);
                    width: 45%">
                    <br>
                    Figure 2. Inactivated virus preservation solution 
                    <br>
                </div>
                </p>
                <br>
                <p>
                    &nbsp;
                    In this project, inactivated virus is used for subsequent nucleic acid extraction and protein
                    purification. Because inactivated virus not only retains the viral nucleic acid fragment, but also
                    destroys the high-level structure of viral protein, which makes it no longer have physiological
                    activity, thus losing the ability of infection, pathogenicity and reproduction. However,
                    conventional
                    inactivation does not affect the primary structure of viral protein, indicating that it can still be
                    specifically recognized.
                    <br>
                    <br>
                </p>
                <br>
                <br>
                <h4 style="color: rgb(237, 88, 58)"> 2. Cell lysis</h4>
                <br>
                <p>
                    &nbsp;
                    Before the characteristic protein detection of samples, the clinical samples need cell lysis and
                    virus inactivation. Cell lysis is an important experimental method to change the permeability and
                    activity of cells. It includes chemical cleavage, enzymatic cleavage and mechanical cleavage.
                    Chemical cleavage and enzymatic cleavage are usually mild methods, usually with little disruption of
                    nucleic acids. These two methods (including SDS and lysozyme treatment) are commonly used in DNA
                    extraction and purification. Compared with chemical lysis, mechanical treatment has higher
                    efficiency and lower selectivity. Mechanical treatment can cleave cells more intensively and
                    comprehensively, but it can also cause nucleic acid fragmentation.
                    <br>
                    <br>
                    &nbsp;
                    Based on the requirements of hardware design, chemical cracking requires less hardware, so we use
                    chemical cracking method to extract our detection marker N protein. The main purpose of cell lysate
                    is as follows: (1) to destroy lipid bilayer by detergent; (2) to dissolve protein; (3) to stabilize
                    protein denaturation; (4) to inhibit protease activity.
                    <br>
                    <br>
                    &nbsp;
                    Here are the reagents and their functions in cell lysis: 50 mm Tris HCl, pH 7.4 (buffer system), 150
                    mm NAC (isotonic system), 1 mm PMSF (powerful protease inhibitor), 1 mm EDTA (denaturant and
                    stabilizer), 5 μ g / ml aprotinin (protease inhibitor), 5 μ g / ml leupeptin (protease inhibitor),
                    1% Triton X-100 (cell destruction), 1% sodium deoxyphosphate (moderate denaturant and egg white
                    dissolving agent), 0.1% SDS (strong denaturant and protein dissolving agent). 7 M urea, 2 m thiourea
                    (can improve the dissolution of membrane protein), protease K, etc.
                    <br>

                </p>
                <br>
                <br>
                <h4 style="color: rgb(237, 88, 58)"> 3. Protein extraction and purification </h4>
                <br>
                <p>
                    &nbsp;
                    Proteins constitute the majority of the virus mass, and the proteins in virus particles can be
                    combined with sugars and lipids to form glycoproteins or lipoproteins. Viral structural proteins can
                    be obtained from purified viral products by separating them from nucleic acids and other proteins
                    without breaking peptide bonds. Neutral detergents, polar reagents or protective agents, such as
                    urea, guanidine hydrochloride, sodium deoxycholate and SDS, should be used for separation, or the pH
                    value should be changed.
                    <br>
                    <br>
                    &nbsp;
                    Extraction and purification method of characteristic protein (i.e. aqueous solution extraction
                    method): the aqueous solution of dilute salt buffer system has good stability and high solubility
                    for protein, and is the most commonly used dropping agent for protein extraction. The dosage is
                    usually 1-5 times of the volume of raw materials, and it needs to be evenly stirred during
                    extraction to facilitate the dissolution of protein. The temperature of extraction depends on the
                    nature of active ingredients. On the one hand, the solubility of most proteins increases with the
                    increase of temperature. High temperature is conducive to dissolution and shortens extraction time.
                    On the other hand, protein denaturation and inactivation will be caused by the increase of
                    temperature. Therefore, based on this point, low temperature (below 5 degrees) operation is
                    generally used for protein and enzyme extraction. In order to avoid degradation during protein
                    extraction, proteinase inhibitors (such as diisopropyl chlorophosphoric acid, iodoacetic acid, etc.)
                    can be added. In this method, we need to consider the following two points: 1. PH value: protein and
                    enzyme are amphoteric electrolytes with isoelectric point, and the pH value of extraction solution
                    should be selected in the pH range deviating from both sides of isoelectric point. When extracting
                    with dilute acid or alkali, it is necessary to prevent the cross reaction of protein dissociative
                    groups caused by excessive acid or alkali, which will lead to irreversible change of protein
                    conformation. Generally speaking, alkaline protein is extracted with acidic extract, while acidic
                    protein is extracted with alkaline extract. 2. Salt concentration: dilute concentration can promote
                    the dissolution of protein, known as salt dissolution. At the same time, due to the partial
                    combination of salt ions and protein, Wu has the advantage of protecting protein from denaturation.
                    Therefore, a small amount of neutral salt such as NaCl is added into the extraction solution,
                    generally 0.15 mol. It is better to increase the concentration. The buffer solution is usually
                    0.02-0.05m phosphate and carbonate isotonic solution. We used aqueous solution extraction method to
                    separate and purify the protein from the sample preservation solution treated by cell lysis, and the
                    sample protein solution with low impurity concentration could be obtained,
                    <br>
                    <br>
                    &nbsp;
                    In order to ensure the safety and convenience of the experiment, we will use the self-made new
                    coronary pneumonia N protein to test the feasibility of the whole experimental process:
                    <br>
                </p>
                <br>
                <h5 style="color: rgb(237, 88, 58)"> 3.1 sequence acquisition of N protein and transformation of
                    recombinant plasmid </h5>
                <br>
                <p>

                    &nbsp;
                    Through literature search, the sequence information of N protein was determined on NCBI (Fig. 3).
                    After
                    the template was synthesized by reagent company, PCR amplification was performed, and it was
                    connected
                    to plasmid pET28a with his * 6 tag. We used heat shock transformation to construct engineering
                    bacteria.
                    Plasmids are small circular double stranded DNA, which can reduce their volume by super helix
                    folding,
                    so they can easily pass through small holes in the cell membrane. In the low osmotic solution of
                    CaCl2
                    at 0 ℃, the competent cells swell into spheres, and the DNA in the transformation mixture forms a
                    hydroxyl calcium phosphate complex which is resistant to DNase (DNase) and adheres to the cell
                    surface.
                    After short time heat shock treatment at 42 ℃, the cells can contract rapidly, absorb DNA complex
                    and
                    complete transformation. E.coli DH5 α strain was selected as the host. After a tube of DH5 α
                    competent
                    cells were frozen and thawed on ice, a certain amount of treated pET28a was added under aseptic
                    conditions, in which 20 ng / PL DNA was added into 1pl. Then, after the mixture was gently mixed,
                    two
                    controls were set up at the same time: one was added with 10NG known plasmid under the same
                    conditions
                    as the positive control, and the titer of competent cells and the number of colonies transformed
                    from
                    each PG plasmid DNA could also be detected at the same time. Then, in another experimental group,
                    LPL
                    sterile water was added as negative control, and heat shock for 90 s was carried out under the
                    condition
                    of 42C water bath. Then, after heat shock, the tube was transferred to the ice bath, and the tube
                    was
                    allowed to stand for 2-5 min under the ice bath condition. Then, 600 PL LB liquid medium was added
                    into
                    the tube and incubated for 45 min at 200 R / min at 37 ℃. Finally, according to the appropriate
                    volume
                    gradient, the bacterial solution was spread on the solid LB medium containing kanamycin, and the
                    single
                    colony containing recombinant plasmid was obtained. Single colony was selected from the plate and
                    inoculated into LB liquid medium containing kanamycin for activation culture.
                    <br>
                </p>
                <br>
                <div align="center">
                    <img src="https://2020.igem.org/wiki/images/4/44/T--BIT--BIO_3.png" style="box-shadow: 0px 0px 4px rgba(0, 0, 0, .5);
             width: 80%">
                    <br>
                    <br>
                    Figure 3. Sequence information of N protein
                </div>
                <br>
                <br>
                <h5 style="color: rgb(237, 88, 58)"> 3.2 induced expression of N protein</h5>
                <br>
                <p>
                    &nbsp;
                    The activated E.coli DH5 α (pET28a) was inoculated into LB liquid medium containing kanamycin and
                    cultured at 37 ℃ to od0.8-1.0. IPTG was added to induce the expression of N protein, and the culture
                    was continued at 37 ℃.4. Primary purification of N protein.
                    <br>
                </p>
                <br>
                <br>
                <h4 style="color: rgb(237, 88, 58)">4. Primary purification of N proteinH</h4>
                <br>
                <p>
                    &nbsp;
                    The culture broth was centrifuged at 4 ℃, 12000r / min to collect the bacteria, and then the
                    bacteria were resuspended with 4 ℃ solid binding buffer (10 mmol / L imidazole, 0.5 mol / L NaCl, 20
                    mmol / L Tris HCl, pH = 7.9), and then placed on the ice to be ultrasonic broken until the
                    suspension became clear; at 4 ℃, 12000r / min, the supernatant was collected from the center,
                    filtered by 0.45 μ m microporous membrane, and then slowly added to Ni In the agarose gel column,
                    the column was then washed with Soluble column of 15 times the volume of Binding Buffer to wash out
                    the impurity protein. Then the column was cleaned by Elution Buffer (500 mmol/L imidazole, 0.5 mol/L
                    NaCl, 20 mmol/L Tris -HCl, and 7.9 = 7.9). After that, the high concentration imidazole was removed
                    by 10kDa ultrafiltration tube, and then the purified N protein was obtained by dissolving PBS again.
                    <br>
                </p>
                <a name="2222"></a>
                <br>
                <br>
                <br>
                <br>
                <br>
                <h3 style="color: rgb(237, 88, 58)">2.2.2.2 aptamer replacement</h3>
                <p>
                    &nbsp;
                    In this step, we use aptamer replacement system to convert the N protein signal to nucleic acid
                    signal
                    through complementary chain replacement for subsequent related experiments.
                </p>
                <br>
                <h4 style="color: rgb(237, 88, 58)">1. Principle</h4>
                <br>
                <h5 style="color: rgb(237, 88, 58)">1.1．Screening</h5>
                <br>
                <p>
                    &nbsp;
                    Aptamer is a single stranded oligonucleotide fragment (DNA or RNA) screened by in vitro exponential
                    enrichment system evolution technology. It is isolated from a random sequence library containing
                    1012-15 different oligonucleotides. This separation process is called exponential enrichment ligand
                    system evolution (SELEX). The screening process is as follows: an initial DNA library is incubated
                    with the target molecules that need to be bound. The unconjugated DNA molecules are discarded, and
                    the bound DNA chains are recycled, amplified by PCR, and injected into a new round of screening.
                    <br>
                    <br>
                    &nbsp;
                    SELEX is the abbreviation of exponential enrichment ligand system evolution. Its basic principle is
                    to construct a synthetic single chain random oligonucleotide library by using molecular biology
                    technology. Its random sequence length is about 20-100 bases. The target ssDNA / ssRNA complex was
                    formed by mixing the target material with the random library. The unconjugated aptamers were washed
                    off and the nucleic acid molecules bound to the target material were separated. The latter was used
                    as a template for PCR amplification to carry out the next round of screening process. Through
                    multiple rounds of cycle screening, the nucleic acid molecules that are not bound or low affinity
                    binding to the target molecules are removed, while the aptamers with high affinity to the target
                    molecules are isolated, and the purity increases with the screening process. If the reverse
                    transcription step is added after each screening cycle of SELEX, RNA aptamers can be screened. The
                    selected aptamers were synthesized in vitro for target recognition, experimental diagnosis and
                    treatment research.
                    <br>
                </p>
                <br>
                <div align="center">
                    <img src="https://2020.igem.org/wiki/images/3/39/T--BIT--BIO_4.png" style="box-shadow: 0px 0px 4px rgba(0, 0, 0, .5);
             width: 80%">
                    <br>
                    <br>
                    Figure 4. Basic principle of SELEX technology
                </div>
                <br>
                <p>
                    &nbsp;
                    The separation method in aptamer screening is magnetic bead method, which combines the target
                    material to the magnetic bead and uses the magnetic bead screening. The separation method needs less
                    target molecules, can quickly and efficiently separate the bound or unbound oligonucleotide
                    sequences, and the operation is more simple. The specific process is as follows: through the
                    interaction between streptavidin and biotin, biotinylated target protein is fixed on magnetic beads.
                    After that, the specific binding sequence was isolated, RT-PCR amplification and transcription were
                    completed automatically through the set program. Finally, the screened sequence was cloned into the
                    vector for sequencing and identification.
                    <br>
                </p>
                <br>
                <div align="center">
                    <img src="https://2020.igem.org/wiki/images/3/38/T--BIT--BIO_5.png" style="box-shadow: 0px 0px 4px rgba(0, 0, 0, .5);
                     width: 80%">
                    <br>
                    <br>
                    Figure 5. X-ray crystallographic analysis of streptavidin
                </div>

                <br>
                <h5 style="color: rgb(237, 88, 58)">1.2．Interaction with target molecules</h5>
                <br>
                <p>
                    &nbsp;
                    The chemical nature of aptamers is nucleic acid, and its binding to ligands is based on the
                    diversity of single strand nucleic acid structure and spatial conformation. In the presence of the
                    target molecule, it can form stable three-dimensional spatial structures such as hairpin, false
                    knot, convex ring and G2 tetrad through the pairing of some complementary bases in the chain,
                    electrostatic interaction and hydrogen bonding. The aptamer structure formed in this way has a large
                    contact area with the target molecule, which can bind tightly with the target material and has high
                    affinity and specificity. The aptamers screened repeatedly by SELEX technology combine with target
                    molecules with high affinity and specificity. Aptamers are widely used in the development of various
                    analytical methods due to their small molecular weight, chemical synthesis, macro preparation and
                    stability in a certain range of temperature, solvent and pH. in recent years, aptamers have been
                    reported for disease diagnosis and treatment, food safety testing and environmental monitoring.
                    <br>
                    <br>
                    &nbsp;
                    The basic principle of this method is as follows: (1) the antigen or antibody is bound to the
                    surface of a solid-phase carrier and its immune activity is maintained. (2) Linking an antigen or
                    antibody with an enzyme to form an enzyme labeled antigen or antibody that retains both the immune
                    activity and the enzyme activity. In the process of determination, the tested sample and enzyme
                    labeled antigen or antibody react with the antigen or antibody on the surface of solid phase carrier
                    in different steps. The antigen antibody complex formed on the solid-phase carrier was separated
                    from other substances by washing. Finally, the amount of enzyme bound on the solid-phase carrier was
                    proportional to the amount of the tested substance in the sample. Because of the high catalytic
                    frequency of enzyme, the reaction effect can be greatly amplified, so that the determination method
                    can reach a high sensitivity.
                    <br>
                    <br>
                    &nbsp;
                    Enzyme linked immunosorbent assay can be used to determine antigen and antibody. There are three
                    necessary reagents in this method: (1) solid phase antigen or antibody. (2) An enzyme labeled
                    antigen or antibody. (3) The substrate of enzyme action. According to the source of reagents, the
                    characteristics of specimens and the conditions of detection, we can design different types of
                    detection methods. Our team used avidin biotin ELISA method in this project. Biotin streptavidin
                    signal amplification ELISA is based on the traditional double antibody sandwich ELISA, according to
                    each streptavidin can combine with four biotins to form an amplification system. It is well known
                    that antibodies and biotin can be coupled together by covalent bond, so streptavidinase can react
                    with biotin coupled with antibody, which plays a certain role in signal amplification of
                    experimental results.
                </p>
                <br>
                <br>
                <h4 style="color: rgb(237, 88, 58)">2. Steps</h4>
                <br>
                <h5 style="color: rgb(237, 88, 58)">2.1 steps of aptamer screening</h5>
                <br>
                <p>
                    &nbsp;
                    (1) selection of screening methods: according to the research purpose and target characteristics,
                    the specific SELEX screening method was designed; (2) the establishment of oligonucleotide Library:
                    according to the selected screening method, the oligonucleotide combination library with appropriate
                    length and sufficient capacity was established; (3) circular screening: to select the suitable
                    aptamer characteristics and to design the SELEX screening method After SELEX screening, the aptamers
                    were sequenced and specifically modified to improve the selectivity of aptamers.
                    <br>
                </p>
                <br>
                <h5 style="color: rgb(237, 88, 58)">2.2 steps of SELEX Technology:</h5>
                <br>
                <p>
                    &nbsp;
                    (1) SELEX technology also has some requirements for target materials. For example, single molecule
                    target materials require high purity to reduce nonspecific binding, and it is difficult to screen
                    non-polar and uncharged materials.
                    <br>
                    <br>
                    &nbsp;
                    (2) Construction of random DNA oligonucleotide Library: SELEX first synthesized single stranded
                    random oligonucleotide library according to molecular biology technology. The library usually
                    contains 1014-1015 single stranded oligonucleotide sequences. The two ends of the library were fixed
                    sequences as PCR primers, and the binding sites were random sequences with the length of 20-80 NT.
                    If RNA is screened, T7 promoter sequence and antisense primers should be added at the 5 'end of the
                    library, so that the library can be translated into RNA library and reverse transcription for PCR
                    amplification.
                    <br>
                    <br>
                    &nbsp;
                    (4) When the affinity of the enriched oligonucleotide library is not increased in the screening
                    cycle, the unique aptamers of the target material are screened out after 6-20 screening cycles. In
                    general, we sequence and analyze 50 or more selected clones.
                    <br>
                    <br>
                    &nbsp;
                    (5) In addition, for further application, a functional group of late modified aptamers is added,
                    which can be used to detect and fix target materials.
                    <br>
                </p>
                <br>
                <h5 style="color: rgb(237, 88, 58)">2.3 steps of SELEX Technology:</h5>
                <br>
                <p>
                    &nbsp;
                    In this project, after cell lysis, the aptamer replacement system for N protein of new coronavirus
                    was designed to process protein signal into nucleic acid signal. The specific design is as follows:
                    (1) at one end of aptamer, biotin streptavidin is used to bind the magnetic bead, which can be fixed
                    by magnet, and the aptamer can be bound to its complementary chain through the hydrogen bonding
                    force of base complementary pairing; (2) when there is a target protein in the sample, the protein
                    will combine with the aptamer, and since the force between protein and aptamer is greater than the
                    hydrogen bond force, the aptamer can bind to the aptamer The complementary chain is released to
                    convert protein signal into nucleic acid signal.
                    <br>
                </p>
                <br>
                <br>
                <h4 style="color: rgb(237, 88, 58)">3. Aptamer replacement</h4>
                <br>
                <h5 style="color: rgb(237, 88, 58)">3.1 protein competitive response</h5>
                <br>
                <p>
                    &nbsp;
                    According to the binding length of aptamer and complementary chain, two different complementary
                    chains were designed. The binding length of S1 and aptamer was 15 BP, the sequence was 5 '-
                    fam-cccaaccccctaaaaa, the binding length of complementary chain S2 and aptamer was 25 BP, and the
                    sequence was 5' - fam-cccaaccctaaacactaaagcatcctcccg The optimal complementary chain was determined
                    by detecting the fluorescence intensity of the supernatant after the reaction. The experimental
                    results are shown in the figure.
                    <br>
                </p>
                <div align="center">
                    <img src="https://2020.igem.org/wiki/images/e/e3/T--BIT--BIO_6.png" style="box-shadow: 0px 0px 4px rgba(0, 0, 0, .5);
                     width: 50%">
                    <br>
                    <br>
                    Figure6. fluorescence of the supernatant after competitive reaction of complementary chains with
                    different binding lengths
                </div>
                <br>
                <p>
                    &nbsp;
                    It can be seen from the results in the figure that when the binding length of complementary chain
                    and aptamer is 15 BP, the fluorescence value of the supernatant after competitive reaction is
                    higher, which proves that the competitive reaction effect of CK-MB is more obvious under this
                    binding length. However, considering that the reaction with H1, H2 and competitive reaction with
                    CKMB is better, H1 and H2 reaction probes are designed for long chain complementary chain S2.
                    <br>
                </p>
                <br>
                <h5 style="color: rgb(237, 88, 58)">3.2 the simulation results and sequence information of H1 and H2
                    probes were designed according to S2 complementary chain.</h5>
                <br>
                <p>

                    The first one:
                    <br>
                    N-protein aptamer: GGGGGGTGGGTGGGGGA TCTCGGAGGATGCTTTTAGGGGGTTGGG
                    <br>
                    <span style="color: red;">
                        <span style="background-color: aqua;">CKMB Complementary strand：CCAACC</span><span
                            style="background-color: yellow;">CCCTAAAAGC</span><span
                            style="background-color: blue;">ATCCTCCG</span>
                        <br>
                        <span style="background-color: yellow;">H1:CCCTAAAAG</span><span
                            style="background-color: blue;">CATCCTCCG</span>CAAAGT<span
                            style="background-color: blue;">CGGAGGATG</span><span
                            style="background-color: yellow;">CTTTTAGGG</span><span
                            style="background-color: greenyellow;">GGTTGG</span>
                        H2:CTTTG<span style="background-color: blue;">CGGAGGATG</span><span
                            style="background-color: yellow;">CTTTTAGGG</span><span
                            style="background-color: greenyellow;">CCAACC</span><span
                            style="background-color: yellow;">CCCTAAAAG</span><span
                            style="background-color: blue;">CATCCTCCG</span>
                    </span>
                    <br>
                    The concentrations of H1 and H2 were 1 μ m and 0.05 μ m, respectively
                </p>
                <br>
                <div align="center">
                    <img src="https://2020.igem.org/wiki/images/4/4d/T--BIT--BIO_7.png" style="box-shadow: 0px 0px 4px rgba(0, 0, 0, .5);
                     width: 50%">
                    <br>
                    <br>
                    Figure7. Binding of complementary chain with H1H2
                </div>
                <br>
                <br>
                <br>
                <div align="center">
                    <img src="https://2020.igem.org/wiki/images/d/d7/T--BIT--BIO_8.png" style="box-shadow: 0px 0px 4px rgba(0, 0, 0, .5);
                         width: 50%">
                    <br>
                    <br>
                    Figure8. Binding structure of complementary chain and H1H2
                    <br>
                    Free energy of complex formed by complementary chain and H1H2
                </div>
                <br>
                <p>
                    The second type:<br>
                    N protein aptamers: GCTGGATGTCACCGGATTGTCGGACATCGGATTGTCTGAGTCATATGACACATCCAGC<br>
                    Complementary strand：GCTGGATGTGTCATATGACTCAGA<br>
                    H1：TGTGTCATATGACTCAGACAAAGTCTGAGTCATATGACACATCCAGC<br>
                    H2：CTTTGTCTGAGTCATATGACACAGCTGGATGTGTCATATGACTCAGA<br>
                    The concentrations of H1 and H2 were 1 μ m and 0.05 μ m, respectively<br>
                </p>
                <br>
                <div align="center">
                    <img src="https://2020.igem.org/wiki/images/9/96/T--BIT--BIO_9.png" style="box-shadow: 0px 0px 4px rgba(0, 0, 0, .5);
                         width: 50%">
                    <br>
                    <br>
                    Figure9. Binding of complementary chain with H1H2
                </div>
                <br>
                <br>
                <br>
                <div align="center">
                    <img src="https://2020.igem.org/wiki/images/0/0d/T--BIT--BIO_10.png" style="box-shadow: 0px 0px 4px rgba(0, 0, 0, .5);
                         width: 50%">
                    <br>
                    <br>
                    Figure10. Binding structure of complementary chain and H1H2
                    <br>
                    Free energy of complex formed by complementary chain and H1H2
                </div>
                <a name="2223"></a>
                <br>
                <br>
                <br>
                <br>
                <br>
                <h3 style="color: rgb(237, 88, 58)">2.2.2.3 isothermal amplification of RCA and HCR</h3>
                <br>
                <p>
                    &nbsp;
                    In this step, we used RCA and HCR technology to amplify the complementary chain of aptamer obtained
                    in the previous step, so as to facilitate the subsequent detection of lateral chromatography strip.
                    <br>
                </p>
                <br>
                <h4 style="color: rgb(237, 88, 58)">1. Principle of RCA Technology</h4>
                <br>
                <p>
                    &nbsp;
                    Rolling circle amplification (RCA) can perform relatively infinite single strand amplification of
                    single stranded circular DNA at the same temperature, without the need for a specific thermal
                    cycling apparatus, and the exponential amplification can be achieved by using 1-2 primers per
                    strand. There are many kinds of RCA detection methods, but basically follow the following two basic
                    principles. One is based on the fact that the rolling loop can only be replicated at the same
                    temperature only when the loop of single strand DNA is closed and there are corresponding primers.
                    The principle of RCA related technology is that rolling ring replication is only used as a cascade
                    amplification system of signals, and the specific nucleic acid or protein can be detected by using
                    pre designed closed loop template and primers coupled with antigenic substances or specific
                    antibodies .
                    <br>
                    <br>
                    &nbsp;
                    The key of RCA reaction is to construct a complete single stranded DNA loop for subsequent
                    amplification. The methods of RCA formation include viscous end loop and terminal loop. RCA can be
                    divided into linear RCA, exponential RCA, multi primer RCA and immune RCA. At present, rolling loop
                    amplification technology is widely used in nucleic acid sequencing, whole genome DNA detection,
                    single nucleotide polymorphism (SNPs) and protein analysis.
                    <br>
                </p>
                <br>
                <h4 style="color: rgb(237, 88, 58)">2. Operation steps of RCA Technology</h4>
                <br>
                <p>
                    &nbsp;
                    The key to the RCA reaction process is the success of cyclization and the high efficiency of
                    amplification. In the laboratory stage, the specificity of the probe was characterized by 2% agarose
                    gel electrophoresis, and the type and amount of ligase, the amount of polymerase and dNTPs were
                    compared and optimized by strip test.
                    <br>
                </p>
                <br>
                <h5 style="color: rgb(237, 88, 58)">2.1 establishment of RCA reaction system</h5>
                <br>
                <p>
                    &nbsp;
                    In this experiment, locked DNA was used to form a loop. The reaction principle is shown in Fig. 6.
                    The cyclization reaction system was 5 μ L, 200 nm miRNA 1 μ L, 200 nm lock probe 1 μ L, splint r
                    ligase 1.5 u, 10 × reaction buffer 0.5 μ L, and the rest was supplemented with DCEP water. The
                    enzyme was inactivated at 37 ℃ for 2 h and 65 ℃ for 20 min. Cool naturally to room temperature.
                    <br>
                    <br>
                    &nbsp;
                    Linear RCA amplification method was used in this experiment. The reaction principle is shown in
                    Figure 7. The rest of the PCR products were amplified by PCR at 30 μ L, 30 μ L, 0.65 μ L, 10.5 μ L,
                    10.5 μ L, 10.5 μ L, 10.5 μ L, respectively. Cool naturally to room temperature.
                    <br>
                </p>
                <br>
                <div align="center">
                    <img src="https://2020.igem.org/wiki/images/e/ea/T--BIT--Bio_P.jpg" style="box-shadow: 0px 0px 4px rgba(0, 0, 0, .5);
                         width: 80%">
                    <div align="center">Figure 11. Schematic diagram of locked DNA loop formation </div>
                </div>
                <br>
                <div align="center">
                    <img src="https://2020.igem.org/wiki/images/3/32/T--BIT--Bio_Q.jpg" style="box-shadow: 0px 0px 4px rgba(0, 0, 0, .5);
                         width: 80%">
                    <div align="center">Figure 12. Schematic diagram of RNA amplification </div>
                </div>
                <br>
                <p>
                    &nbsp;
                    The RCA experimental products were characterized by 2% agarose gel electrophoresis. D2000 was used
                    as the Marker of the experiment. When agarose gel electrophoresis was used, the 1-6 swimming passes
                    from D2000 to Maker were followed by blank, sample, sample, sample, blank, and the experimental
                    results were shown in Figure 3. According to the experimental results, we can conclude that the RCA
                    amplification is successful because of the long chain amplified by RCA. Therefore, it can be proved
                    that the RCA amplification is successful from the band of 2-5 crossing.
                    <br>
                </p>
                <br>
                <div align="center">
                    <img src="https://2020.igem.org/wiki/images/b/b2/T--BIT--BIO_12.png" style="box-shadow: 0px 0px 4px rgba(0, 0, 0, .5);
                         width: 60%">
                    <br>
                    <br>
                    Figure 13. RCA experimental results
                    <br>
                </div>
                <br>
                <h4 style="color: rgb(237, 88, 58)">3. Advantages of RCA Technology</h4>
                <br>
                <p>
                    &nbsp;
                    With the rapid development of molecular biology, nucleic acid isothermal amplification technology
                    has been widely used in biological detection, such as hybrid chain reaction (HCR), catalytic hairpin
                    assembly (CHA), and strand displacement amplification, SDA) and loop mediated isothermal
                    amplification (lamp). These isothermal amplification techniques can be divided into enzyme free
                    amplification and enzyme consumption amplification according to the conditions of enzyme use. Enzyme
                    free amplification included HCR, CHA, SDA and LAMP. Compared with enzyme-free amplification, RCA
                    reaction is simpler in design, and the amplification product ssDNA can be directly combined with the
                    probe to achieve signal amplification. The product is a long ssDNA, the product is relatively
                    single, and the reaction does not need too many primers.
                    <br>
                    <br>
                    &nbsp;
                    As a rapid and constant temperature DNA amplification technology, a large number of ssDNA amplified
                    by RCA can be used to carry a series of signal molecules to amplify the detection signal. It has
                    been widely used in microbial detection, disease prevention and environmental monitoring. Therefore,
                    novel coronavirus pneumonia was detected by RCA amplification in this experiment.
                    <br>
                </p>
                <br>
                <h4 style="color: rgb(237, 88, 58)">4. Principle of HCR Technology</h4>
                <br>
                <p>
                    &nbsp;
                    Hybridization chain reaction uses competitive hybridization between nucleic acid chains as energy
                    source to realize signal amplification, which can be carried out at room temperature without the
                    assistance of temperature change and other enzymes. The mechanism is shown in Fig. 8. The hairpin
                    probes H1 and H2 are designed. H1 consists of four parts: A, B, C and B *. Among them, B and B *
                    have 18 pairs of base pairs complementary to form a double chain as the H1 stem; C is the ring part
                    of the hairpin structure, which is a single chain containing 6 bases; a is the single strand viscous
                    end extending from the hairpin stem; H2 includes four parts: C *, B, a * and B *, and its design is
                    similar to H1. A, B, C can be respectively the same as a *, b *, and B *, respectively, C *
                    complementary. When there is no target DNA (initiator, a * b *), although H1 and H2 have some
                    complementary chains, they can exist stably in the solution due to 18 base pairs in the stem of
                    hairpin structure, and they will not cross each other and open each other; When the initiation DNA
                    strand a * b * is added, a * b * first complements the stem and sticky end ab of H1, and the
                    competition causes H1 to be opened, revealing that the CB * single strand can open the hairpin
                    structure H2, thus exposing a * b * with the same sequence as the target DNA. H1 and H2 can continue
                    to open H1 in turn, and H1 and H2 will open the hairpin structure H1 and H2 after the hybridization
                    reaction initiated by the target DNA, A long double stranded DNA polymer with a gap formed by
                    alternating hybridization of H1 and H2 is formed to amplify the signal greatly and realize the
                    purpose of signal amplification.
                    <br>
                    <br>
                    &nbsp;
                    In this experiment, we designed H1 and H2 probes corresponding to the complementary chain of aptamer
                    to achieve amplification.
                    <br>
                </p>
                <br>
                <div align="center">
                    <img src="https://2020.igem.org/wiki/images/d/d9/T--BIT--BIO_13.png" style="box-shadow: 0px 0px 4px rgba(0, 0, 0, .5);
                         width: 80%">
                    <br>
                    <br>
                    Figure 14. Schematic diagram of HCR amplification
                    <br>
                </div>
                <br>
                <h4 style="color: rgb(237, 88, 58)">5. Advantages of HCR Technology</h4>
                <p>
                    &nbsp;
                    Hybrid chain reaction (HCR) is a kind of chain replacement (tmsd) reaction based on toehold. It is
                    caused by its fast reaction speed, high sensitivity, super specificity, certain sample tolerance,
                    structural flexibility, enzyme free property, isothermal condition, independent instrument, simple
                    operation method and excellent amplification efficiency (the chain replacement rate can be increased
                    by 10e6 times by tmsd) It has attracted great interest. Especially, the whole process of HCR
                    technology does not require the participation of enzymes. This advantage makes HCR technology get
                    rid of the harsh reaction conditions of the enzyme, so that it can be amplified in a relatively
                    loose experimental environment (such as medium strong acid, medium strong alkali, etc.), which
                    improves the success rate of amplification reaction, and has good reproducibility. At the same time,
                    the requirements for the instrument are greatly simplified. Only simple temperature control
                    equipment can complete the anti reaction It should be that the cost of the experiment can be greatly
                    reduced by both. HCR is an isothermal and non enzyme - promoted amplification strategy, unlike
                    traditional PCR analysis. Another important difference between HCR and PCR is that HCR is a probe
                    amplification technology, and PCR is a target amplification technology. Therefore, HCR can
                    effectively reduce the false positive results and cross contamination of amplification in PCR.
                    <br>

                </p>
                <br>
                <h4 style="color: rgb(237, 88, 58)">6. Experimental operation and result analysis of HCR technology</h4>
                <br>
                <p>
                    &nbsp;
                    According to the complementary chain obtained by aptamer replacement in the previous step, we
                    designed two kinds of H1 and H2 probes for HCR reaction, and verified their binding effect with
                    different complementary chains by experiments.
                    <br>
                </p>
                <br>
                <h5 style="color: rgb(237, 88, 58)">6.1 HCR experimental procedures and control conditions used</h5>
                <br>
                <p>
                    &nbsp;
                    The hairpin probes H1 and H2 were diluted with tris HCl buffer (1 μ m, 100 μ L), heated in boiling
                    water for 5 minutes and cooled at room temperature for 1 hour. At the same time, the target was
                    diluted to 0.1 μ M. H1, H2 and RP were mixed into 30 μ L system at room temperature according to the
                    volume ratio of 1:1:1 at room temperature, and were used for nucleic acid chain hybridization after
                    2 hours.
                    <br>
                    <br>
                    (1) Type and concentration of buffer solution
                    <br>
                    Tris HCl buffer (20mm Tris HCl + 0.5m NaCl, pH = 8.0)
                    <br>
                    0.01M PBS buffer (pH = 7.4)
                    <br>
                    Commercial 20xssc buffer solution (set 20x, 10x, 5x and 1x groups respectively)
                    <br>
                    <br>
                    (2) Buffer pH
                    <br>&nbsp;
                    According to the results of Experiment 1, the type and concentration of buffer were determined, and
                    then the pH value of buffer was adjusted.
                    <br>
                    For Tris HCl buffer solution, the pH groups were set at 7.5, 8.0 and 8.5, respectively
                    <br>&nbsp;
                    For 0.01M PBS buffer, the pH groups were set at 7.0, 7.4 and 8.0, respectively
                    <br>
                    <br>
                    (3) Ionic strength
                    <br>&nbsp;

                    According to the results of Experiment 1 and 2, the type, concentration and pH of buffer solution
                    were determined, and the ion strength was changed by adding magnesium ion into the buffer system.
                    The magnesium ion concentration groups were set as 0,10,50,100 mM MgCl2 respectively
                    <br>
                    <br>
                    (4) Reaction time
                    <br>
                    &nbsp;
                    On the basis of Experiment 1, 2 and 3, the reaction time groups were set as 0.5 h, 1 h, 1.5 h and 2
                    h, respectively
                    <br>
                    <br>
                    (5) Reaction temperature
                    <br>
                    &nbsp;
                    On the basis of Experiments 1, 2, 3 and 4, the reaction temperature groups were set as 4 ℃, 25 ℃
                    (room temperature), 37 ℃ and 65 ℃, respectively.
                    <br>
                    <br>
                    (6) H1 and H2 length
                    <br>
                    &nbsp;
                    On the basis of Experiments 1, 2, 3, 4 and 5, the chain length groups were set as 32, 42 (original
                    length) and 52 BP respectively.
                    <br>
                </p>
                <br>
                <h5 style="color: rgb(237, 88, 58)">6.2 Experimental results and analysis of HCR Technology</h5>
                <br>
                <p>
                    &nbsp;
                    According to the experimental results, we can conclude that there are electrophoresis bands because
                    the long strands of nucleic acid have been amplified from HCR, which proves that the HCR
                    amplification is successful.
                </p>
                <div align="center" style="box-shadow: 0px 0px 0px transparent;">
                    <img src="https://2020.igem.org/wiki/images/b/bc/T--BIT--BIO_14.png"
                        style="box-shadow: 0px 0px 0px transparent; width: 80%;">
                    <br>
                    Figure 15. HCR experimental results
                    <br>
                </div>
                <a name="2224"></a>
                <br>
                <br>
                <br>
                <br>
                <br>
                <h3 style="color: rgb(237, 88, 58)">2.2.2.4 nucleic acid signal detection</h3>
                <br>
                <p>
                    &nbsp;
                    The last part of the biological module of this project is to detect the amplified DNA fragments, and
                    to achieve the purpose of sample detection by analyzing the detection results. After full
                    investigation and preliminary experiment in the laboratory, we decided to use the lateral
                    chromatography strip technology to select colloidal gold as a marker for detection, and in the
                    follow-up through visual observation and mobile phone module linkage mode for more accurate
                    quantitative detection.
                </p>
                <br>
                <h4 style="color: rgb(237, 88, 58)">1. Lateral tomography</h4>
                <br>
                <p>
                    &nbsp;
                    The basic principle of lateral chromatography lfca is to fix specific antigen, antibody or nucleic
                    acid probe on NC membrane with large pore diameter (NC membrane, nitrocellulose membrane) as
                    carrier. When the sample to be tested is added to the sample pad at one end of the test strip, it
                    moves laterally through capillary action, and has specific immune reaction with colloidal gold or
                    fluorescent microsphere labeled reagent on the binding pad It should be moved to the NC membrane and
                    captured by the antigen, antibody or nucleic acid probe fixed on the surface of the NC membrane, and
                    gathered on the detection band. The visual color development results can be obtained by visually
                    measuring the light reflection signal density of nitrocellulose surface markers (colloidal gold or
                    fluorescent latex particles). Other unconjugated markers cross the detection zone and flow into the
                    absorbent pad to achieve the purpose of automatic separation.
                </p>
                <br>
                <h4 style="color: rgb(237, 88, 58)">2. Immune colloidal gold technology</h4>
                <br>
                <p>
                    &nbsp;
                    Immune colloidal gold technology (gict) is a kind of immunolabelling technology which uses colloidal
                    gold as a tracer marker to apply to antigen and antibody. Colloidal gold labeling is essentially a
                    coating process in which proteins and other macromolecules are adsorbed on the surface of colloidal
                    gold particles. Colloidal gold is a visible light color signal, which has negative charge in weak
                    alkali environment. It can form a strong electrostatic binding with the positive charge group of
                    protein molecule, and does not affect the biological characteristics of protein. In addition to
                    proteins, colloidal gold can also bind with many other biological macromolecules, such as spa, PHA,
                    ConA, etc. Therefore, we can use the physical properties of colloidal gold to label some bioactive
                    substances as immune probes, which can be used for immunohistochemical antigen localization, antigen
                    and anti physical examination. At the binding site of the gold labeled protein, dark brown particles
                    were observed under the microscope. When a large number of these markers gather at the corresponding
                    ligands, red or pink spots will be visible to the naked eye. Therefore, colloidal gold immunoassay
                    can be used in qualitative or semi quantitative rapid immunoassay. At the same time, the preparation
                    of colloidal gold antibody is very simple and low cost, which has high application value in this
                    project.
                </p>
                <br>
                <h4 style="color: rgb(237, 88, 58)">3. Construction and implementation of test strip</h4>
                <br>
                <p>
                    &nbsp;
                    Lfca is mainly composed of the components (antigen, antibody, protein, nucleic acid, etc.) in the
                    liquid mixture formed by dissolving the detected substance in the solvent on the reaction matrix
                    based on the immune recognition or nucleic acid hybridization and antibody labeling technology
                    through the difference of moving speed of capillary force. Based on the above principle, we
                    constructed the following lateral chromatography strip. The strip consists of four parts: sample
                    pad, binding pad, chromatography membrane and absorbent pad. As shown in the figure below, a simple
                    side flow chromatographic strip is formed by superimposing these four parts on the support plate.
                    The sample pad is a treated fiber membrane or glass wool, which is used to quickly absorb the sample
                    to be tested and make it flow laterally to the binding pad by capillary action; the binding pad is a
                    fiber membrane or glass wool, which adsorbs the labeled bioactive material, which can combine with
                    the detection target in the sample solution to form an immune complex (or nucleic acid hybrid chain)
                    visible to the naked eye; Most chromatographic membranes are nitrocellulose (NC) membrane, which is
                    the key material in side flow analysis technology. It provides a platform for the reaction between
                    analytes. There are two or more different bioactive substances (such as antigen, antibody and
                    nucleic acid probe), test line (t line) and control line (C line) printed on it. It is used to
                    intercept the labeled immune complex (or nucleic acid hybrid chain) and visually display the
                    detection results; the absorbent pad is an absorbent paperboard, which is used to absorb the samples
                    to be tested through the chromatographic membrane, so as to balance the pressure difference on both
                    sides of the chromatographic membrane, and promote more samples to flow laterally on the
                    chromatographic membrane.

                </p>
                <br>
                <div align="center">
                    <img src="https://2020.igem.org/wiki/images/f/f6/T--BIT--BIO_X.png" style="box-shadow: 0px 0px 4px rgba(0, 0, 0, .5);
                         width: 80%">
                    <br>
                    <br>
                    Figure 16. Structure of lateral chromatographic strip 
                    <br>
                </div>
                <br>
                <p>
                    &nbsp;
                    At the same time, we used lateral flow nuclear acid biosensors (lfnabs). The lateral chromatographic
                    nucleic acid sensor is the basis of the traditional lateral chromatography immunoassay strip_ A
                    novel lateral chromatography technique based on nucleic acid probe for target detection was
                    developed. Because the colloidal gold test strip is used this time, different from the traditional
                    immunochromatographic strip, the detection line and quality control line of lfnabs are generally
                    fixed with a biotin modified capture DNA probe at one end. The designed sulfhydryl modified DNA is
                    usually connected to the colloidal gold, and the DNA in the gold labeled complex and the capture
                    probe on the detection line are paired by base complementary or The gold nanoparticles can be
                    trapped and retained on the detection line.
                    <br>
                    <br>
                    &nbsp;
                    In this project, c-probe and T-line capture probe corresponding to specific fragment of target
                    nucleic acid were fixed at different positions on NC membrane. One end of t-probe could be
                    complementary with corresponding partial base of target nucleic acid. On the other hand, colloidal
                    gold can be labeled with corresponding probe to form a gold labeled complex, which can be applied to
                    the gold label pad. One end of the detection probe can also be complementary with the other end of
                    the target nucleic acid. When there is a target in the sample to be tested, with the migration of
                    the sample on the test strip, the target nucleic acid will be partially hybridized with the
                    detection probe in the gold label complex on the gold label pad. As the sample drives the gold label
                    complex to continue to migrate on the NC membrane, the target nucleic acid in the sample will be
                    partially hybridized with the corresponding t probe, The sandwich structure of t-probe microRNA (or
                    microdna) - SH probe is formed to capture and retain gold nanoparticles in the corresponding
                    detection line position. When the target in the sample reaches a certain concentration, the more
                    gold nanoparticles are captured, the stronger the color of the line, and the red band can be
                    observed by naked eyes; If there is no target in the sample to be tested, the "sandwich" sandwich
                    structure can not be formed, and the gold nanoparticles can not stay and gather on the detection
                    line, so the red band can not be observed in the detection area by naked eyes.
                </p>
                <br>
                <div align="center">
                    <img src="https://2020.igem.org/wiki/images/b/ba/T--BIT--BIO_16.png" style="box-shadow: 0px 0px 4px rgba(0, 0, 0, .5);
                         width: 60%">
                    <br>
                    <br>
                    Figure 17. Colloidal gold pad <br>
                </div>
                <br>
                <br>
                <h4 style="color: rgb(237, 88, 58)">4. Experimental results and analysis of Colloidal gold Technology
                </h4>
                <br>
                <p>
                    &nbsp;
                    We explore the effect of temperature on HCR reaction through experiments, and characterize the
                    amplification results with lateral chromatography strips.
                </p>
                <br>
                <div align="center">
                    <img src="https://2020.igem.org/wiki/images/4/4b/T--BIT--BIO_17.png" style="box-shadow: 0px 0px 4px rgba(0, 0, 0, .5);
                                 width: 60%">
                    <br>
                    <br>
                    Figure 18. Left 1 and 2 are blank groups, Right 1 and 2 are experimental groups (0.1 μ m) <br>
                </div>
                <br>
                <br>
                <div align="center">
                    <img src="https://2020.igem.org/wiki/images/b/b9/T--BIT--BIO_18.png" style="box-shadow: 0px 0px 4px rgba(0, 0, 0, .5);
                                 width: 60%">
                    <br>
                    <br>
                    Figure 19. Left 1 and 2 are blank groups, Middle 1 and 2 are experimental groups (0.01 μ m), Right 1 and 2 are
                    experimental groups (0.1 μ m)
                    <br>
                </div>
                <br>
                <a name="3"></a>
                <br>
                <br>
                <br>
                <br>
                <br>
                <br>
                <h2 style="color: rgb(237, 88, 58)"> 2.2.3 analysis of innovation points</h2>
                <br>
                <h3 style="color: rgb(237, 88, 58)"> 2.2.3.1 application of aptamer replacement</h3>
                <br>
                <p>
                    &nbsp;
                    The novel coronavirus detection is a task with a large amount of accuracy and high accuracy.
                    According to the background and related literature, we have two kinds of detection paths. One is the
                    use of nucleic acid signals, the other is the use of protein signals. Only nucleic acid detection
                    will have higher requirements for laboratory personnel, and it is easy to produce false negative
                    results due to various situations; only protein detection can only be qualitative detection, which
                    is easy to produce false positive results due to various situations. Therefore, our project team has
                    considered this aspect. We should use both nucleic acid signal and protein signal of virus, so we
                    consider using aptamer for signal conversion The nucleocapsid protein (N protein) signal was
                    converted into nucleic acid signal, and the combined detection of nucleic acid and protein signal
                    was carried out. The innovation part mainly includes the following points.
                    <br>
                    <br>
                    &nbsp;
                    (1) In this project, we use aptamer replacement system to convert the N protein signal of new
                    coronavirus into nucleic acid signal for detection, and use the protein signal of new coronavirus.
                    However, the detection through nucleic acid detection is more simple and easy to operate than
                    nucleic acid detection and immune method respectively, and the cost is more considerable, and the
                    detection accuracy is also higher It is more accurate than only nucleic acid detection, reducing the
                    probability of false negative or false positive.
                    <br>
                    <br>
                    &nbsp;
                    (2) In the past, compared with antibody, which is a protein affinity reagent, the application of
                    aptamers has been limited due to the slow development of technology for screening aptamers with high
                    affinity to a certain protein. However, after gradual development, we have mastered the screening
                    methods and techniques of aptamers, and their affinity is equivalent to high-quality antibodies, and
                    more than antibodies It can be hybridized with complementary sequences, which improves the
                    sensitivity and accuracy of detection.
                    <br>
                    <br>
                    &nbsp;
                    (3) Through aptamer replacement system, we can convert a small number of protein sample signals into
                    accounting signals, and then carry out isothermal amplification to obtain a large number of nucleic
                    acid signals, which solves the problem of less protein samples in previous detection.
                    <br>
                </p>
                <br>
                <h3 style="color: rgb(237, 88, 58)">2.2.3.2 comparison of HCR and RCA isothermal amplification methods
                </h3>
                <br>
                <p>
                    &nbsp;
                    HCR and RCA were used in the amplification method. The detection accuracy of new coronavirus was
                    improved by comparing with other amplification methods.
                </p>

                <a name="4"></a>
                <br>
                <br>
                <br>
                <br>
                <br>
                
                <h3 style="color: rgb(237, 88, 58)">3. References</h3>

                <p style="text-indent: 2rem;">
                    [1]Zeng, W. et al. Biochemical characterization of SARS-CoV-2 nucleocapsid 
                    protein. Biochem Biophys Res Commun 527, 618-623, doi:10.1016/j.bbrc.2020.
                    04.136 (2020).
                </p>
                <br>
                <p style="text-indent: 2rem;">
                    [2]	Xu, X. et al. Evolution of the novel coronavirus from the ongoing Wuhan 
                    outbreak and modeling of its spike protein for risk of human transmission. 
                    Sci China Life Sci 63, 457-460, doi:10.1007/s11427-020-1637-5 (2020).
                </p>
                <br>
                <p style="text-indent: 2rem;">
                    [3]	Lu, R. J. et al. Genomic characterisation and epidemiology of 2019 
                    novel coronavirus: implications for virus origins and receptor binding. 
                    Lancet 395, 565-574, doi:10.1016/s0140-6736(20)30251-8 (2020).
                </p>
                <br>
                <p style="text-indent: 2rem;">
                    [4]	Zu, F. et al. The quenching of the fluorescence of carbon dots: 
                    A review on mechanisms and applications. Microchimica Acta 184, 
                    1899-1914, doi:10.1007/s00604-017-2318-9 (2017).
                </p>
                <br>
                <p style="text-indent: 2rem;">
                    [5]	Parashar, N. C., Poddar, J., Chakrabarti, S. & Parashar, G. 
                    Repurposing of SARS-CoV nucleocapsid protein specific nuclease resistant 
                    RNA aptamer for therapeutics against SARS-CoV-2. Infect Genet Evol 85, 
                    104497, doi:10.1016/j.meegid.2020.104497 (2020).
                </p>
                <br>
                <p style="text-indent: 2rem;">
                    [6] Journal of China Pharmaceutical University, Qiao. J. Principle of 
                    rolling circle  amplification and its application in medicine. 37, 201-205. 
                    (2006).
                </p>
                <br>
                <p style="text-indent: 2rem;">
                    [7] Xu Jie et al. Research on label-free fluorescence detection of DNA 
                    based on DNA hybridization chain reaction and hybridization gaps. 046, 
                    1095-1101 (2018).
                </p>
                <br>
                <p style="text-indent: 2rem;">
                    [8] Yang Mei, Zhang Xiaotian, Liu Xiumin, Wu Jing & China Laboratory Diagnosis, 
                    Zhu. J. Application status of new coronavirus detection methods. (2020).
                </p>
                <br>
                <p style="text-indent: 2rem;">
                    [9]Yang Chan et al. Application of Hybridization Chain Reaction in the Design 
                    of Biosensing Probes. 37, 8-16 (2017).
                </p>
            </div>

            <div id="foot-box"
                    style="background-color: rgb(34, 34, 34); text-align: center; color: rgb(255, 255, 255);">
                    <br>
                    Created By Beijing Institute of Technology
                    <br>
                    <br>
            </div>
        </article>

        <a href="#0" class="cd-top">Back Top</a>
    </div>


    <style>
        body {
            background: url(https://2020.igem.org/wiki/images/0/06/T--BIT--bg1.png);
        }

        * {
            margin: 0;
            padding: 0;
        }

        body .inner {
            padding: 20px;
        }

        body .inner h1 {
            font-family: 'Poppins', sans-serif;
            font-size: 3rem;
        }

        body .inner .router a {
            text-decoration: none;
            color: rgb(120, 120, 120);
        }

        body .inner .router a:hover {
            color: rgb(31, 160, 180);
        }

        #main-content {
            width: 90%;
            margin: 0 auto;
        }

        nav {
            width: 25%;
            float: left;
        }

        nav .inner {
            width: 18%;
            margin: 0;
            position: fixed;
            /* height: 100%; */
            background: rgb(251, 251, 251);
            box-shadow: 0 0 10px rgba(0, 0, 0, .5);
            padding: 10px;
        }

        nav .inner .contact {
            background-color: rgb(34, 34, 34);
            padding: 10px;
        }

        nav .inner .nav-bar {
            background-color: rgb(34, 34, 34);
            padding: 10px;
        }

        nav .hidden {
            height: 1px;
        }

        nav ul li {
            /*去掉列表项目的符号*/
            list-style-type: none;
        }

        nav ul a {
            text-decoration: none;
            color: rgb(0, 0, 0);
        }

        nav ul a:hover {
            color: rgb(31, 160, 180);
        }

        article {
            width: 65%;
            float: left;
            background: rgb(251, 251, 251);
            margin: 0;
            padding: 0;
            box-shadow: 0px 0px 20px rgba(0, 0, 0, .5);
        }

        article .inner {
            padding: 40px;
        }


        /*推断页面已经帮我们调了一些间距*/
        #main-content h1 {
            line-height: 3rem;
        }

        #main-content p {
            font-size: .9rem;
            line-height: 2rem;
            font-family: 'Poppins', sans-serif;
        }

        #main-content h2 {
            font-family: 'Poppins', sans-serif;
        }
    </style>
    <script type="text/javascript"
        src="https://2020.igem.org/wiki/index.php?title=Template:BIT/js/jquery-3.5.1.min.js&action=raw&ctype=text/javascript"></script>

    <!--这是返回顶部的js代码，单独抽离之后会失效-->
    <script>
        jQuery(document).ready(function ($) {
            var offset = 300,
                offset_opacity = 1200,
                scroll_top_duration = 700,
                $back_to_top = $('.cd-top');


            $(window).scroll(function () {
                ($(this).scrollTop() > offset) ? $back_to_top.addClass('cd-is-visible') : $back_to_top.removeClass('cd-is-visible cd-fade-out');
                if ($(this).scrollTop() > offset_opacity) {
                    $back_to_top.addClass('cd-fade-out');
                }
            });


            $back_to_top.on('click', function (event) {
                event.preventDefault();
                $('body,html').animate({
                    scrollTop: 0,
                }, scroll_top_duration
                );
            });
        });
    </script>
</body>

</html>
