{{Poster}}

<html>

<style>
    #content {
        background: url(https://2020.igem.org/wiki/images/0/06/T--BIT--bg1.png);
    }

    #section_title {
        margin: 0;
        padding: 2rem;
        background-color: rgb(251, 251, 251);
    }

    #section_text {
        margin: 0;
        padding: 2rem;
        background-color: rgb(251, 251, 251);
        min-height: 1320px;
    }
</style>
<!--这是返回顶部控件的css,不要修改，单独抽离后会失效-->
<style>
    .cd-top {
        display: inline-block;
        height: 40px;
        width: 40px;
        position: fixed;
        bottom: 40px;
        right: 10px;
        box-shadow: 0 0 10px rgba(0, 0, 0, 0.05);
        /* image replacement properties */
        overflow: hidden;
        text-indent: 100%;
        white-space: nowrap;
        background: rgba(232, 98, 86, 0.8) url(https://2020.igem.org/wiki/images/0/0f/T--BIT--Poster_cd-top-arrow.svg) no-repeat center 50%;
        visibility: hidden;
        opacity: 0;
        -webkit-transition: opacity .3s 0s, visibility 0s .3s;
        -moz-transition: opacity .3s 0s, visibility 0s .3s;
        transition: opacity .3s 0s, visibility 0s .3s;
    }

    .cd-top.cd-is-visible,
    .cd-top.cd-fade-out,
    .no-touch .cd-top:hover {
        -webkit-transition: opacity .3s 0s, visibility 0s 0s;
        -moz-transition: opacity .3s 0s, visibility 0s 0s;
        transition: opacity .3s 0s, visibility 0s 0s;
    }

    .cd-top.cd-is-visible {
        /* the button becomes visible */
        visibility: visible;
        opacity: 1;
    }

    .cd-top.cd-fade-out {
        /* if the user keeps scrolling down, the button is out of focus and becomes less visible */
        opacity: .5;
    }

    .no-touch .cd-top:hover {
        background-color: #e86256;
        opacity: 1;
    }

    @media only screen and (min-width: 768px) {
        .cd-top {
            right: 20px;
            bottom: 20px;
        }
    }

    @media only screen and (min-width: 1024px) {
        .cd-top {
            height: 60px;
            width: 60px;
            right: 30px;
            bottom: 30px;
        }
    }
</style>


<div class="page_wrapper">
    <div class="page_center">


        <!---Place the content of your poster inside this div--->
        <div class="poster_w content">


            <div class="row">
                <div class="section full first_text" id="title">
                    <img src="https://2020.igem.org/wiki/images/2/23/T--BIT--Poster_poster_title.jpg">

                    <div class="info">
                        <div class="title">
                            Dr. Watson：j.o.h.n:judge of COVID- 19 handling
                            <br>
                            <br>
                            <em>Presented by BIT 2020</em>
                        </div>
                        <div class="title">
                            
                        </div>
                        <div class="text">
                            <h2>Abstract</h2>
                            <br>
                            <br>
                            <br>
                            Since the outbreak of the COVID-19 at the end of 2019, it has caused great harm to 
                            public health safety. Although the current detection methods are diverse, most of 
                            the main applications are hospitals with certain qualifications and conditions. 
                            This is no small challenge for people in developed, developing, and underdeveloped 
                            countries. Therefore, it is very necessary to have a simple operation and 
                            easy-to-carry testing equipment that can be used in communities and rural hospitals. 
                            Therefore, this year’s BIT team is committed to designing related integrated 
                            detection systems. Based on the designed biosensing module, centrifugal chip 
                            module， smart phone module, and the COVID-19 related protein markers （Convert 
                            it into a nucleic acid signal） are fast, sensitive, portable and low-cost. 
                            Testing, providing effective testing programs for the new coronavirus in places 
                            with low medical testing resources.
                        </div>
                    </div>
                </div>
            </div>


            <div class="row">

                <div class="section half" id="introduction">
                    <!--Here is the image used for the third width section, the size is:  400 x 200 pixels  -->

                    <img src="https://cdn.jsdelivr.net/gh/a-layfolk/FigureBed/img/blog/competition/igem/20201027205606.jpg"
                        alt="">

                    <div class="info">

                        <div class="title">Introduction </div>

                        <div class="text">
                            Our project is dedicated to designing an integrated detection system which can be 
                            used for the new coronavirus to solve the problems of high detection cost and complex 
                            detection process. Based on the biosensing module, the centrifugal chip module and 
                            the smart phone module, we can effectively implement the fast, sensitive, portable 
                            and low-cost sample-to-answer detection of nucleic acid and protein markers related 
                            to coronavirus, which can provide effective coronavirus detection scheme for the 
                            places with scarce medical testing resources.
                            <br>
                            <br>
                            <br>
                            The biosensing module undertakes the function of converting biological signals into 
                            optical signals. The target protein to be detected can be obtained by pre-processing 
                            the respiratory tract extract of the subject. Through the designed nucleic acid 
                            aptamer system, the protein signal can be converted into a nucleic acid signal, 
                            and then the nucleic acid signal can be amplified by an isothermal amplification 
                            system. Finally, we use colloidal gold chromatography test strip technology to 
                            detect the amplified nucleic acid sequence and display the detection results in 
                            the form of light signals.
                            <br>
                            <br>
                            <br>
                            The centrifugal chip module undertakes the function of providing a process carrier 
                            for the biosensing module on the micro-scale.Using the "fidget spinner" method, we 
                            can manually provide the centrifugal force that drives the fluid in the chip. In 
                            the pre-processing area of the chip, we can complete the pre-processing of the 
                            sample, and then provide a constant temperature in the isothermal amplification 
                            area to achieve nucleic acid signal amplification. In the signal detection and 
                            transduction area, we will pre-embed chromatographic test strips for detection to 
                            detect and convert the amplified nucleic acid sequence. The mobile phone support 
                            module provides the colloidal golden degree detection for the above-mentioned 
                            biosensing module. The software designed will implement the human-computer 
                            interaction of the terminal, provide the user with the final result delivery, 
                            and further serve as an intermediate medium for the extension of the back-end 
                            information chain.
                            <br>
                            <br>
                            <br>
                            Based on the above, with the cooperation of the three modules, the project has 
                            achieved the final goal, that is to build a simple and portable, low-cost, 
                            high-utility, multi-use standardization, extended information of the open virus 
                            detection platform and can be applied to the community, rural hospitals and families 
                            of the visual detection hardware equipment.
                        </div>
                    </div>
                </div>

                <div class="section half" id="inspiration">

                    <img src="https://cdn.jsdelivr.net/gh/a-layfolk/FigureBed/img/blog/competition/igem/20201027205606.jpg">


                    <div class="info">

                        <div class="title">Inspiration</div>
                        <div class="text">
                            This is inspiration
                        </div>
                    </div>
                </div>

            </div>



            <div class="row">

                <div class="section full" id="project">
                    <img
                        src="https://cdn.jsdelivr.net/gh/a-layfolk/FigureBed/img/blog/competition/igem/20201027205606.jpg">

                    <div class="info">
                        <div class="title">Project&nbsp;(&nbsp;Methodology, Results and Conclusions&nbsp;)</div>
                        <div class="text">
                            Our program, Dr. Waston, is designed to directly test the N protein for detection. 
                            In our project, We convert protein markers into nucleic acid signals. We use microfluidic 
                            chips to amplify nucleic acid signals，and use colloidal gold test strips for testing. 
                            Finally, the results are analyzed through the supporting smartphone app. In the end, our 
                            project has formed two parts: Biosensor and Hardware.
                            <br>
                            <br>
                            <br>
                            See Bio and Hardware below for details.
                        </div>
                    </div>
                </div>

            </div>




            <div class="row">

                <div class="section half" id="bio">
                    <img src="https://2020.igem.org/wiki/images/6/60/T--BIT--Poster_BIO.png" alt="">

                    <div class="info">
                        <div class="title">BIO</div>
                        <div class="text">
                            The purpose of the biological module is to construct a rapid, stable, high-accuracy
                            and high-sensitivity detection method for the new coronavirus markers, and finally
                            realize the real-time detection of the new coronavirus. At the same time, the biological
                            module undertakes the function of converting biological signals into optical signals.
                            The biological part of this project is mainly divided into four parts: sample
                            pre-processing,
                            nucleic acid aptamer replacement, RCA isothermal amplification and nucleic acid signal
                            detection. The following figure is the experimental flow chart, that is, the obtained N
                            protein generates complementary chains by competing with the aptamer, and then the nucleic
                            acid signal is amplified by the RCA isothermal amplification technology, and then
                            transferred
                            to the test strip for colloidal gold detection. According to the above scheme, we designed
                            relevant pre-experiments for simulation verification in the laboratory stage. For details,
                            see the subsidiary section of the BIO section.
                            <br>
                            <br>
                            <br>
                            <img src="https://2020.igem.org/wiki/images/9/93/T--BIT--Poster_BIO_1.png" alt="">
                        </div>
                    </div>
                </div>


                <div class="section half" id="hardware">
                    <img src="https://2020.igem.org/wiki/images/5/50/T--BIT--Poster_hardware.jpg" alt="">

                    <div class="info">
                        <div class="title">Project Overview</div>
                        <div class="text">
                            In our project, a microfluidic chip driven by a fidget spinner is used as a reaction carrier
                            to
                            realize the biological part. The manufacturing material is polymethyl methacrylate (PMMA),
                            and
                            the chip is assembled by hot-key bonding method. The overall frame of our microfluidic chip
                            can
                            be approximately regarded as a rectangle, the turning radius is slightly smaller than the
                            average
                            length of the adult hand gap, and there is a rotating shaft composed of a roller bearing and
                            a gasket
                            in the center of the chip. By holding the gasket in the center and rotating the chip with
                            the other
                            hand, the centrifugal force generated by the rotation can be used to drive the liquid path
                            of the
                            microfluidic chip.

                            <br>
                            <br>
                            <br>

                            According to the biological part of the detection process, on the basis of the microfluidic
                            chip,
                            we successively realize the isothermal amplification of nucleic acid in the isothermal
                            amplification
                            area, and realize the visual signal conversion based on the lateral chromatography test
                            strip in the
                            signal detection area. Finally, the mobile phone support module optically detects the signal
                            on the test
                            strip, and performs subsequent signal processing. Specifically, on the test strip, our
                            project adopts a
                            signal conversion method based on colloidal gold technology, and compares the colloidal gold
                            degree with
                            the standard positive result through the color reaction to obtain the reaction result.

                            <br>
                            <br>
                            <br>

                            The temperature control module is mainly composed of a single-chip microcomputer, a ceramic
                            heater and a
                            thermistor. The combination of these three can meet our temperature requirements for RCA
                            reactions.

                            <br>
                            <br>
                            <br>

                            In the optical signal visual detection part, we use the MINA framework to build a WeChat
                            mini program to
                            take pictures and upload the colloidal gold test strips, as well as reading the colloidal
                            gold image with
                            the software, and fitting the relationship between the image characteristics and the
                            concentration of the
                            test object, and finally get the virus test results.

                            <br>
                            <br>
                            <br>

                            <img src="https://2020.igem.org/wiki/images/8/8f/T--BIT--Poster_BIO_2.jpg" alt="">
                            <div align="center" style="font-size: .8rem;">
                                Figure. General design of hardware
                            </div>
                        </div>
                    </div>
                </div>

            </div>



            <div class="row">

                <div class="section quarter" id="sampale_pretratment">
                    <img src="https://2020.igem.org/wiki/images/2/28/T--BIT--Poster_Sample_pretratment.png">
                    <div class="info">
                        <div class="title">sample processing</div>
                        <div class="text">
                            <p>
                                The new coronavirus is a positive-stranded single-stranded RNA virus with a genome 
                                of approximately 30kd. It is an RNA virus with a larger genome, which mainly 
                                infects human alveolar epithelial cells. In order to obtain pure N protein of the 
                                new coronavirus, sample pre-processing is divided into three steps: specimen 
                                collection, cell lysis, and protein extraction and purification.
                            </p>
                            <br>
                            <br>
                            <p>
                                In specimen collection, we use common oropharyngeal swabs, nasopharyngeal swabs, or 
                                patients’ deep cough sputum as samples for testing. After collection, this project 
                                uses an inactivated virus (Figure 1) for subsequent protein purification.
                            </p>
                            <br>
                            <br>
                            <!--图片-->
                            <div align="center" >
                                <img src="https://2020.igem.org/wiki/images/a/a8/T--BIT--BIO_2.png">
                            </div>
                            <br>
                            <div align="center" style="font-size: .8rem;">
                                Figure 1 Inactivated virus preservation solution
                            </div>
                            <br>
                            <br>
                            <p>
                                In cell lysis, based on the requirements of hardware design, because chemical lysis 
                                has lower hardware requirements, we use chemical lysis to extract the detection 
                                marker N protein. Mainly the lysate treatment method. The main purposes of the 
                                cell lysate are as follows:. (1) Use detergent to destroy the lipid bilayer and 
                                rupture the cell; (2) Solubilize protein; (3) Denature the protein to stabilize 
                                it (4) Inhibit protease activity. The following are the reagents and their effects 
                                in cell lysis: 50 mM Tris-HCl pH 7.4 (buffer system), 150 mM NaC (isotonic system), 
                                1 mM PMSF (powerful protease inhibitor), 1 mM EDTA (denaturant and stabilizer) 
                                Agent), 5 μg/ml Aprotinin (protease inhibitor), 5 μg/ml Leupeptin (protease 
                                inhibitor), 1% Triton X-100 (destroy cells), 1% Sodium deoxycholate (moderate 
                                denaturant and protein solubilizer), 0.1% SDS (strong denaturant and proteolytic 
                                agent). 7M urea, 2M thiourea (can improve the melting of membrane proteins), 
                                proteinase K, etc.
                            </p>
                            <br>
                            <br>
                            <p>
                                In protein extraction and purification, the virus structural protein can generally 
                                be separated by using neutral detergents, polar reagents, or protective agents that 
                                prevent protein polymerization. In order to ensure the safety and convenience of 
                                the experiment, we will use the self-made new type of coronary pneumonia N protein 
                                to test the feasibility of the entire experimental process. The experimental steps 
                                are divided into the sequence acquisition of the N protein, the transformation of 
                                the recombinant plasmid and the induced expression of the N protein. We searched 
                                the literature and determined the sequence information of the N protein on NCBI 
                                (Figure 2). After synthesizing the template in the reagent company, we performed 
                                PCR amplification and ligated it to the plasmid pET28a with the His*6 tag, and then 
                                Use genetic engineering related methods to transform the recombinant plasmid and 
                                induce the expression of N protein.
                            </p>
                            <br>
                            <br>
                            <div align="center" >
                                <img src="https://2020.igem.org/wiki/images/4/44/T--BIT--BIO_3.png">
                            </div>
                            <br>
                            <div align="center" style="font-size: .8rem;">
                                Figure 2 Sequence information of N protein
                            </div>
                            <br>
                            <br>
                        </div>
                    </div>
                </div>

                <div class="section quarter" id="aptamer_replacement">
                    <img src="https://2020.igem.org/wiki/images/c/c6/T--BIT--Poster_aptamer_replacement.png">
                    <div class="info">
                        <div class="title">Nucleic acid aptamer replacement</div>
                        <div class="text">
                            <p>
                                After obtaining the pure N protein of the new coronavirus, we use a nucleic acid 
                                aptamer replacement system to convert the non-amplifiable N protein signal into an 
                                amplifiable nucleic acid signal in the form of a complementary strand.
                            </p>
                            <br>
                            <br>
                            <p>
                                After obtaining highly specific nucleic acid aptamers, we designed an aptamer 
                                replacement system for the N protein of the new coronavirus to convert protein 
                                signals into nucleic acid signals. The specific design is: use biotin-streptavidin 
                                to bind magnetic beads on one end of the nucleic acid aptamer, which can be fixed 
                                by the magnet, and the aptamer can bind its complement through the hydrogen bond 
                                force of base complementary pairing When there is a target protein in the sample, 
                                the protein will bind to the aptamer, and since the force between the protein and 
                                the aptamer is greater than the hydrogen bond force, the complementary chain is 
                                released and the protein signal is converted into a nucleic acid signal. 
                            </p>
                            <br>
                            <br>
                            <!--图片-->
                            <div align="center" >
                                <img src="https://2020.igem.org/wiki/images/7/75/T--BIT--Poster_aptamer_01.png"
                                     style="box-shadow: 0px 0px 4px rgba(0, 0, 0, .5);
                                 width: 70%"
                                >
                            </div>
                            <br>
                            <br>
                            <p>
                                In order to verify the feasibility of the experiment, we used CKMB protein and its 
                                aptamer to carry out the above experiments. According to the binding length of the 
                                aptamer and complementary chain of CKMB, we designed two different complementary 
                                chains. The binding length of the complementary chain S1 and the aptamer is 15 bp, 
                                and its sequence is 5'-FAM-CCCAACCCCCTAAAA, complementary chain The binding length 
                                of S2 and the aptamer is 25 bp, and its sequence is 5'-FAM-CCCAACCCCCTAAAAGCATCCTCCG. 
                                Set the CK-MB protein concentration to 2000 ng/mL and the incubation temperature to 
                                37°C. Perform a competition reaction experiment and detect the supernatant after 
                                the reaction. Fluorescence intensity of the liquid to determine the best 
                                complementary chain. The experimental results are shown in Figure 2.
                            </p>
                            <br>
                            <br>
                            <div align="center" >
                                <img src="https://2020.igem.org/wiki/images/e/e3/T--BIT--BIO_6.png"
                                     style="box-shadow: 0px 0px 4px rgba(0, 0, 0, .5);
                                 width: 70%"
                                >
                            </div>
                            <br>
                            <h4 align="center">
                                Figure 2 The fluorescence of the supernatant after the competition reaction of complementary strands with different binding lengths
                            </h4>
                            <br>
                            <br>
                            <p>
                                It can be seen from the results in the figure that when the binding length of the 
                                complementary chain and the aptamer is 15 bp, the fluorescence value of the 
                                supernatant after the competition reaction is higher, which proves that the 
                                competitive reaction effect of CK-MB is more obvious under this binding length.
                            </p>
                            <br>
                            <br>
                        </div>
                    </div>
                </div>

                <div class="section quarter" id="microfluidic——chip">
                    <img src="https://2020.igem.org/wiki/images/1/10/T--BIT--Poster_microfluidic_chip.jpg">
                    <div class="info">
                        <div class="title">Microfluidic chip</div>
                        <div class="text">
                            <p>
                                Our design of centrifugal chip realizes the HCR, RCA nucleic acid amplification, 
                                make the colloidal gold strip fixed, and struct a fidget spinner model. 
                            </p>
                            <br>
                            <br>
                            <p>
                                Our chip uses the PMMA material and adopts the structure of a spinner fidget, on 
                                which nucleic acid amplification chamber and Colloidal gold strip chamber are 
                                constructed. As a material that is widely used in industry, the processing and 
                                manufacturing process of PMMA has been mature, and its processing method is simple 
                                and easy and the micro-manufacturing cost is low. 
                            </p>
                            <br>
                            <br>
                            <p>
                                According to the centrifugal chip layer we designed, we can obtain 1-4 layers of 
                                different structures. In order to ensure the cleanliness of the chamber after the 
                                chip is bonded, the surface is cleaned with absolute ethanol and deionized water 
                                in sequence. After cleaning, wipe off the remaining water stains with lens cleaning 
                                paper, place the chip layers in the glass plate in order, fix each layer of the 
                                chip with solid micro pins, cover another layer of glass plate and clamp it. Place 
                                it in a 170°C environment and heat it for 10 minutes, take out the fixed micro 
                                pins and heat it for 5 minutes to complete the bonding of the chip to obtain the 
                                required microfluidic chip. Finally, place the rolling bearing in the center of 
                                the chip to complete the centrifugal chip processing.
                            </p>
                            <br>
                            <br>
                            <!--图片-->
                            <div align="center" >
                                <img src="https://2020.igem.org/wiki/images/0/0e/T--BIT--HardWare_06.png"
                                     style="box-shadow: 0px 0px 4px rgba(0, 0, 0, .5);
                                 width: 70%"
                                >
                            </div>
                            <br>
                            <h4 align="center">
                                Figure1. Chip surface cleaning process
                            </h4>
                            <br>
                            <br>
                            <div align="center" >
                                <img src="https://2020.igem.org/wiki/images/4/4c/T--BIT--HardWare_07.png"
                                     style="box-shadow: 0px 0px 4px rgba(0, 0, 0, .5);
                                 width: 70%"
                                >
                            </div>
                            <br>
                            <h4 align="center">
                                Figure2. Chip bonding process
                            </h4>
                            <br>
                            <br>
                            <div align="center" >
                                <img src="https://2020.igem.org/wiki/images/0/04/T--BIT--HardWare_08.png"
                                     style="box-shadow: 0px 0px 4px rgba(0, 0, 0, .5);
                                 width: 70%"
                                >
                            </div>
                            <br>
                            <h4 align="center">
                                Figure3. Centrifugal chip physical map
                            </h4>
                            <br>
                            <br>
                            <p>
                                Our project is based on solidworks software to design the chip structure, as shown 
                                in the figure below. The chip is designed with a micro-reaction chamber, a 
                                micro-channel and a hole, and contains two symmetrical detection modules, which 
                                can simultaneously detect two independent samples. The micro device is composed 
                                of four layers of 2 mm PMMA boards, which are connected as a whole by means of 
                                hot-key bonding method. The first layer contains the sample inlet, the second layer 
                                contains part of the nucleic acid amplification chamber and test strip chamber and 
                                microchannels, the third layer contains part of the nucleic acid amplification 
                                chamber and test strip chamber, and the fourth layer is only Test strip chamber. 
                                The sequence of the liquid path is the injection port, the nucleic acid 
                                amplification chamber, and the test strip chamber. The radius of the nucleic acid 
                                amplification chamber is about 5mm. In the nucleic acid amplification chamber, the 
                                sample is subjected to isothermal amplification at a constant temperature (provided 
                                by the mobile phone module device). At the same time, during the actual use of the 
                                RPA nucleic acid amplification area, surface modification treatment is also 
                                required to prevent the reaction mixture from directly contacting the inner surface 
                                of the channel, avoiding the surface adsorption of biological/chemical particles, 
                                and preventing microfluidic interference. Inhibition of nucleic acid amplification 
                                and carryover contamination. The size of the test strip chamber is 3.9mm*59mm, 
                                which is used to store the test strip for subsequent testing operations.
                            </p>
                            <br>
                            <br>
                            <div align="center" >
                                <img src="https://2020.igem.org/wiki/images/7/7c/T--BIT--HardWare_10.png"
                                     style="box-shadow: 0px 0px 4px rgba(0, 0, 0, .5);
                                 width: 50%"
                                >
                                <img src="https://2020.igem.org/wiki/images/5/58/T--BIT--HardWare_09.png"
                                     style="box-shadow: 0px 0px 4px rgba(0, 0, 0, .5);
                                 width: 50%"
                                >
                            </div>
                            <br>
                            <h4 align="center">
                                Figure 4. Chip structure design concept diagram
                            </h4>
                            <br>
                            <br>
                        </div>
                    </div>
                </div>

                <div class="section quarter" id="centrifugal_liquid_drive">
                    <img src="https://2020.igem.org/wiki/images/1/1e/T--BIT--Poster_centrifugal_liquid_drive.jpg   ">
                    <div class="info">
                        <div class="title">Centrifugal liquid drive</div>
                        <div class="text">
                            <p>
                                Centrifugal force drive is a unique technology in microfluidic driving technology, 
                                and its system uses the centrifugal force generated by the chip to make circular 
                                motion as the driving force of the liquid flow. The flow velocity can be adjusted 
                                and controlled by changing the rotation speed of the chip and designing different 
                                channel configurations.
                            </p>
                            <br>
                            <br>
                            <p>
                                The special feature of our centrifugal microfluidic chip is that it does not use a 
                                motor, but uses a pushing hand to drive the microfluidic chip to rotate, so as to 
                                construct a simulated gravity field, that is, the centrifugal field, and the flow 
                                is controlled by the volume force acting on the material in the centrifugal 
                                microfluidic chip, so as to get rid of the complicated laboratory equipment. 
                                Based on the previous model construction, we have produced a centrifugal 
                                microfluidic chip model, as shown in the figure below. Instead of installing 
                                gaskets in this chip, we rotate directly with a solid center shaft. Preliminary 
                                tests show that the chip can maintain a basically stable rotation and speed with 
                                the handheld center shaft. The following is the experimental operation:
                            </p>
                            <br>
                            <br>
                            <p>
                                ①In order to simulate the fluid flow during the actual rotation of the centrifugal chip, we injected 100μL of pigment into the amplification chamber of the chip, and placed a test strip in the chip test strip area in advance. It is evaluated by visually observing the pigment flow. Figure 1  shows that the pigment can be injected normally without entering the test strip area.
                            </p>
                            <br>
                            <br>
                            <!--图片-->
                            <div align="center" >
                                <img src="https://2020.igem.org/wiki/images/8/8f/T--BIT--modelling_1.png"
                                     style="box-shadow: 0px 0px 4px rgba(0, 0, 0, .5);
                                 width: 70%"
                                >
                            </div>
                            <br>
                            <h4 align="center">
                                Figure 1. Pigment injection
                            </h4>
                            <br>
                            <br>
                            <p>
                                ②Turn the chip upside down to deposit the pigment in the amplification chamber on the other side of the chip chamber, and at the same time turn the centrifugal chip to rotate around the central axis. The liquid in the amplification chamber enters the test strip chamber under the action of centrifugal force, and covers the entire test strip under the capillary action of the test strip. By observing the liquid flow during the rotation process and the pigment diffusion of the test strip, it can be proved that the centrifugal force provided by the toggle chip can drive the liquid in the amplification chamber into the detection area of the test strip.
                            </p>
                            <br>
                            <br>
                            <div align="center" >
                                <img src="https://2020.igem.org/wiki/images/6/60/T--BIT--modelling_2.png"
                                     style="box-shadow: 0px 0px 4px rgba(0, 0, 0, .5);
                                 width: 70%"
                                >
                            </div>
                            <br>
                            <h4 align="center">
                                Figure 2. Actual driving situation
                            </h4>
                            <br>
                            <br>
                        </div>
                    </div>
                </div>


            </div>





            <div class="row">
                <div class="section quarter" id="rca">
                    <img src="https://2020.igem.org/wiki/images/3/36/T--BIT--Poster_rca.png">
                    <div class="info">
                        <div class="title">RCA isothermal amplification</div>
                        <div class="text">
                            <p>
                                In this step, we used RCA technology to amplify the complementary chain of aptamer 
                                obtained in the previous step for the convenience of subsequent detection.
                            </p>
                            <br>
                            <br>
                            <p>
                                RCA can be used to amplify single stranded circular DNA at the same temperature. It 
                                does not need a specific thermal cycling apparatus. It can achieve exponential 
                                amplification with 1-2 primers per strand, and it is fast, specific and sensitive. 
                                There are many kinds of RCA detection methods at present. The principle we adopt 
                                is that rolling ring replication is only a kind of cascade amplification system of 
                                signal, using designed closed loop template and primers coupled with antigenic 
                                substance or specific antibody to detect specific nucleic acid or protein. The 
                                following is the operation and results of our RCA amplification experiment.
                            </p>
                            <br>
                            <br>
                            <p>
                                The key to the RCA reaction process is the success of cyclization and the high 
                                efficiency of amplification. In the laboratory stage, the specificity of the probe 
                                was characterized by 2% agarose gel electrophoresis, and the type and amount of 
                                ligase, the amount of polymerase and dNTPs were compared and optimized by strip test.
                            </p>
                            <br>
                            <br>
                            <p>
                                In this experiment, we used locked DNA to form a loop (Fig. 1). The cyclization 
                                reaction system was 5 μ L, 200 nm miRNA 1 μ L, 200 nm lock probe 1 μ L, splint r 
                                ligase 1.5 u, 10 × reaction buffer 0.5 μ L, and the rest was supplemented with DCEP 
                                water. The enzyme was inactivated at 37 ℃ for 2 h and 65 ℃ for 20 min. Cool 
                                naturally to room temperature. Then the amplification reaction was carried out. 
                                Linear RCA amplification method was used in this experiment (Fig. 2). The 
                                amplification reaction system was 10 μ L, cyclization product 5 μ L, phi 29 DNA 
                                polymerase 0.4 u, 10 × reaction buffer 0.5 μ L, dNTPs 300 μ μ μ, and the rest was 
                                supplemented with te buffer, and the enzyme was inactivated by reaction at 37 ℃ 
                                for 2 h and 65 ℃ for 15 min. Cool naturally to room temperature.	
                            </p>
                            <br>
                            <br>
                            <!--图片-->
                            <div align="center" >
                                <img src="https://2020.igem.org/wiki/images/0/03/T--BIT--Poster_RCA_01.jpeg"
                                     style="box-shadow: 0px 0px 4px rgba(0, 0, 0, .5);
                                 width: 70%"
                                >
                            </div>
                            <br>
                            <h4 align="center">
                                Figure 1. principle of DNA loop formation
                            </h4>
                            <br>
                            <br>
                            <div align="center" >
                                <img src="https://2020.igem.org/wiki/images/e/e0/T--BIT--Poster_RCA_02.jpeg"
                                     style="box-shadow: 0px 0px 4px rgba(0, 0, 0, .5);
                                 width: 70%"
                                >
                            </div>
                            <br>
                            <h4 align="center">
                                Figure 2. principle of RNA amplification
                            </h4>
                            <br>
                            <br>
                            <p>
                                In this RCA experiment, D2000 was used as the Marker of the experiment. When 
                                agarose gel electrophoresis was used, the 1-6 swimmers from left to right were 
                                next to D2000 Maker: blank, samples, samples, samples, and blanks. According to 
                                the experimental results, we can conclude that the RCA amplification is successful 
                                because of the long chain amplified by RCA. Therefore, the band of 2-5 crossing 
                                can prove the successful amplification of RCA.
                            </p>
                            <br>
                            <br>
                            <div align="center" >
                                <img src="https://2020.igem.org/wiki/images/b/b2/T--BIT--BIO_12.png"
                                     style="box-shadow: 0px 0px 4px rgba(0, 0, 0, .5);
                                 width: 70%"
                                >
                            </div>
                            <br>
                            <h4 align="center">
                                Figure 3. RCA experimental results
                            </h4>
                            <br>
                            <br>
                        </div>
                    </div>
                </div>

                <div class="section quarter" id="nucleic_acid_signal_detection">
                    <img src="https://2020.igem.org/wiki/images/3/38/T--BIT--Poster_nucleic_acid_signal_detection.png">
                    <div class="info">
                        <div class="title">nucleic acid signal detection</div>
                        <div class="text">
                            <p>
                                Finally, we detected the nucleic acid signal obtained from the previous step through 
                                the lateral chromatography strip technology to obtain the final results. In this 
                                experiment, colloidal gold was selected as a marker for detection, and the 
                                subsequent quantitative detection was realized by visual observation and mobile 
                                phone module linkage. The structure of lateral chromatography strip is as follows:
                            </p>
                            <br>
                            <br>
                            <!--图片-->
                            <div align="center" >
                                <img src="https://2020.igem.org/wiki/images/f/f6/T--BIT--BIO_X.png"
                                     style="box-shadow: 0px 0px 4px rgba(0, 0, 0, .5);
                                 width: 70%"
                                >
                            </div>
                            <br>
                            <h4 align="center">
                                Figure 1. Structure of lateral chromatographic strip
                            </h4>
                            <br>
                            <br>
                            <p>
                                In this project, we use lateral flow nuclear acid biosensors (IFNABs). The lateral 
                                chromatography nucleic acid sensor is a new lateral chromatography technology which 
                                is derived from the traditional lateral chromatography immunoassay strip and uses 
                                nucleic acid probe to detect the target. Because the colloidal gold test strip is 
                                used this time, different from the traditional immunochromatographic strip, the 
                                detection line and quality control line of lfnabs are generally fixed with a biotin 
                                modified capture DNA probe at one end. The designed sulfhydryl modified DNA is 
                                usually connected to the colloidal gold, and the DNA in the gold labeled complex 
                                and the capture probe on the detection line are paired by base complementary or 
                                The gold nanoparticles can be trapped and retained on the detection line.
                            </p>
                            <br>
                            <br>
                            <p>
                                In this project, c-probe and T-line capture probe corresponding to specific fragment 
                                of target nucleic acid were fixed at different positions on NC membrane. One end of 
                                t-probe could be complementary with corresponding partial base of target nucleic 
                                acid. On the other hand, colloidal gold can be labeled with corresponding probe to 
                                form a gold labeled complex, which can be applied to the gold label pad. One end of 
                                the detection probe can also be complementary with the other end of the target 
                                nucleic acid. When there is a target in the sample to be tested, with the migration 
                                of the sample on the test strip, the target nucleic acid will be partially 
                                hybridized with the detection probe in the gold label complex on the gold label 
                                pad. As the sample drives the gold label complex to continue to migrate on the NC 
                                membrane, the target nucleic acid in the sample will be partially hybridized with 
                                the corresponding t probe, The "sandwich" structure of T probe microRNA sh probe 
                                was formed, and the gold nanoparticles were captured and retained in the 
                                corresponding detection line position. When the target in the sample reached a 
                                certain concentration, the red band could be observed by naked eyes. According to 
                                the test results of the test strip, we can get the content of the target nucleic 
                                acid. The experimental results of colloidal gold strip are as follows:
                            </p>
                            <br>
                            <br>
                            <div align="center" >
                                <img src="https://2020.igem.org/wiki/images/b/b9/T--BIT--BIO_18.png"
                                     style="box-shadow: 0px 0px 4px rgba(0, 0, 0, .5);
                                 width: 70%"
                                >
                            </div>
                            <br>
                            <h4 align="center">
                                Figure 2. Left 1 and left 2 are blank groups, middle 1 and 2 are experimental 
                                groups (0.01 μ m), right 1 and right 2 are experimental groups (0.1 μ m)
                            </h4>
                            <br>
                            <br>
                        </div>
                    </div>
                </div>

                <div class="section quarter" id="temprature_control">
                    <img src="https://2020.igem.org/wiki/images/2/24/T--BIT--Poster_temprature_control.jpg">
                    <div class="info">
                        <div class="title">Temperature control</div>
                        <div class="text">
                            Our temperature control module can provide our amplification with stable temperature, the commonly used PCR amplification on the temperature change after heating denaturation and annealing of two steps, which requires the device in the temperature control needs to have the ability of rapid heating and cooling, which means you need larger space to provide equipment heat dissipation of the environment, but also has great power to provide our equipment. However, because of our need for miniaturization and portability, we did not adopt PCR as our amplification technology. Instead, we adopted an amplification method with low requirement for temperature change to meet our demand for portability. This amplification method only needs to be amplified at constant temperature.
                            
                            <br>
                            <br>
                            <br>

                            Since the amplification temperature we needed was constant, we initially considered whether we could use human body to amplify the nucleic acid temperature of our chip. However, considering the biosafety and stable temperature required for amplification, we looked elsewhere. We achieve the temperature control of amplification by a temperature control circuit with a customized heating film. We use the temperature sensor to collect the temperature signal, the microprocessor MSP430 built-in A/D module is used to take sample, software control algorithm is used to calculate the control quantity. Through the I/O port output power, we control MOS tube or load to switch on or off, output constant current by controling hardware circuit, and achieve heating and cooling process. The stability of temperature is guaranteed relying on simple control PID algorithm.

                            <br>
                            <br>
                            <br>

                            <img src="https://2020.igem.org/wiki/images/4/4d/T--BIT--Poster_Temperature_control_1.png" alt="">
                            <div align="center" style="font-size: .8rem;">
                                Figure1. Customized heating film
                            </div>

                            <img src="https://2020.igem.org/wiki/images/d/de/T--BIT--Poster_Temperature_control_2.png" alt="">
                            <div align="center" style="font-size: .8rem;">
                                Figure2. Temperature control circuit
                            </div>

                            <br>
                            <br>
                            <br>

                            The following picture is the image of temperature data fitting using Matlab after the temperature sensor is used to receive the temperature. 
                            
                            <br>
                            <br>
                            <br>

                            <img src="https://2020.igem.org/wiki/images/1/11/T--BIT--Poster_Temperature_control_3.png" alt="">
                            <div align="center" style="font-size: .8rem;">
                                Figure3. Temperature curve of heating process fitted by matlab
                            </div>
                        </div>
                    </div>
                </div>

                <div class="section quarter" id="smartphone_support">
                    <img src="https://2020.igem.org/wiki/images/5/53/T--BIT--Poster_smartphone_support.jpg">
                    <div class="info">
                        <div class="title">Smartphone support</div>
                        <div class="text">
                            For colloidal gold detection, after taking a photo with a mobile phone, we analyze the R value after the image is collected and compare it with the positive R value to determine the  result of the test.
                            
                            <br>
                            <br>
                            <br>

                            Through the establishment of the small program, we have completed the following functions: reading the colloidal gold image, fitting the relationship between image features and solution concentration, outputting the image, and outputting the predicted concentration.

                            <br>
                            <br>
                            <br>

                            Software interactive program module: Based on the development of WeChat mini program, extract features such as image color (RGB), and use the least square method to perform linear fitting to obtain the relationship between features and concentration. Least squares method is a mathematical optimization technique. It finds the best function match of the data by minimizing the sum of squares of errors. The least square method can be used to easily obtain unknown data, and minimize the sum of squares of errors between the obtained data and the actual data. Software interaction process: the mobile phone shoots multiple images, the mini program reads the images, manually enters the solution concentration, image processing, fitting the relationship between the image characteristics and the solution concentration, the new image is input in the mini program, and the predicted concentration is output.
                            
                            <br>
                            <br>
                            <br>

                            <img src="https://2020.igem.org/wiki/images/8/8d/T--BIT--Poster_Smartphone_support_1.png" alt="">
                            <div align="center" style="font-size: .8rem;">
                                Figure1. WeChat mini program initial page
                            </div>

                            <img src="https://2020.igem.org/wiki/images/5/53/T--BIT--Poster_Smartphone_support_2.jpeg" alt="">
                            <div align="center" style="font-size: .8rem;">
                                Figure2. Interface page
                            </div>

                            <br>
                            <br>
                            <br>

                            To eliminate the influence of imaging differences between different phones, we provide 6 standard colloidal gold test strips. After tapping the start button，users should first upload more than 3 pictures of standard strips to generate a standard curve only for the specific smartphone. Then upload the strip picture we want to be tested and get the result.

                            <br>
                            <br>
                            <br>

                            For the detection of the results, we use mobile phones to take photos and upload the collected images to WeChat mini program. After reading the colloidal gold images,

                            <br>
                            <br>
                            <br>

                            The least square method was used to fit the relation between image color and solution concentration through WeChat mini program, and the final output predicted concentration.


                        </div>
                    </div>
                </div>

            </div>



            <div class="row">
                <div class="section full" id="human_practice">
                    <img src="https://2020.igem.org/wiki/images/6/65/T--BIT--Poster_Human_practice.jpg">

                    <div class="info">
                        <div class="title">human Practice</div>
                        <div class="text">
                            Although this year was affected by the COVID-19, the BIT team still carried out a lot of creative work in three aspects: background research, Program improvement and education.
                        </div>
                    </div>

                </div>
            </div>



            <div class="row">
                <div class="section third" id="background_research">
                    <img src="https://2020.igem.org/wiki/images/6/6d/T--BIT--Poster_Background_research.png">

                    <div class="info">
                        <div class="title">Background research</div>
                        <div class="text">
                            After thorough investigation, we have determined to make a highly accurate, simple and portable new coronavirus detection device, which should be able to help hospitals in communities, rural areas and other places as well as large-scale population detection and investigation. Therefore, we determined the target audience based on the characteristics of the project, conducted interviews and surveys with the target audience, and asked them their suggestions for our project.

                            <br>
                            <br>
                            <br>

                            Their opinions helped us determine that our project can help solve real-world problems.

                            <br>
                            <br>
                            <br>

                            <img src="https://2020.igem.org/wiki/images/f/f9/T--BIT--Poster_Background_research_2.png" alt="">
                        </div>
                    </div>

                </div>

                <div class="section third" id="program_improvement">
                    <img src="https://2020.igem.org/wiki/images/1/10/T--BIT--Poster_Program_improvement.png">

                    <div class="info">
                        <div class="title">Program improvement</div>
                        <div class="text">
                            In the middle of the project, we participated in a number of iGEMer gatherings and had in-depth exchanges with the teams we were cooperating with. We received many suggestions to help us improve the project. At the same time, we also interviewed companies and gained a lot of inspiration. After we upgraded the project, we visited the doctors we interviewed and got their further suggestions.
                            
                            <br>
                            <br>
                            <br>

                            <img src="https://2020.igem.org/wiki/images/4/4f/T--BIT--Poster_Program_improvement_1.png" alt="">
                            
                        </div>
                    </div>

                </div>

                <div class="section third" id="education">
                    <img src="https://2020.igem.org/wiki/images/a/a2/T--BIT--Poster_Education.png">

                    <div class="info">
                        <div class="title">Education</div>
                        <div class="text">
                            In this year's popular science publicity work, the BIT team has done a lot of work with "online publicity" and " Team inheritance" as its work goals.

                            <br>
                            <br>
                            <br>

                            In terms of online publicity, the BIT team summarized past online work experience and wrote a research report, hoping to help more teams in popularizing science. At the same time, in order to form a bigger brand so that our science popularization work can help more people, we united the other two teams to jointly establish the "Science for All Alliance", hoping to make the current science popularization work better .

                            <br>
                            <br>
                            <br>

                            In Team inheritance, BIT team created "course to iGEM" and "Student to teacher", hoping to attract more students to become iGEMer. At the same time, we have prepared a series of courses and lectures to help them become more professional iGEMer.

                            <br>
                            <br>
                            <br>

                            <img src="https://2020.igem.org/wiki/images/3/38/T--BIT--Poster_Education_1.png" alt="">
                        </div>
                    </div>

                </div>

            </div>




            <div class="row">
                <div class="section full" id="collabration_and_partnership">
                    <img src="https://2020.igem.org/wiki/images/4/45/T--BIT--Poster_Collaboration_and_partnership.jpg">

                    <div class="info">
                        <div class="title">Collabration and partnership</div>
                        <div class="text">
                            Although it is difficult for us to enter the laboratory due to the impact of the COVID-19,
                            we still cooperated with other teams in other ways and benefited each other's projects.
                            <h2>
                                BIT-China：
                            </h2>
                            We reached a consensus on team inheritance and jointly held a series of activities including
                            providing suggestions for each other’s projects and co-hosting lectures.
                            <h2>
                                SCUT-China:
                            </h2>
                            Our two teams reached a cooperation and conducted a summary of online science popularization
                            experience and future prospects in different aspects.And also, we founded the Science for
                            All Alliance for common long-term goals and sustainable development of iGEMers' science
                            communication.
                            <h2>
                                UESTC-China:
                            </h2>
                            we founded the Science for All Alliance for common long-term goals and sustainable
                            development of iGEMers' science communication.
                            <h2>
                                SCU-China:
                            </h2>
                            We have provided a lot of material for the human practice of SCU-China.

                            <img src="https://2020.igem.org/wiki/images/5/58/T--BIT--Poster_collaboration_1.png" alt="">
                        </div>

                    </div>
                </div>
            </div>




            <div class="row">
                <div class="section third" id="implementation">
                    <img
                        src="https://cdn.jsdelivr.net/gh/a-layfolk/FigureBed/img/blog/competition/igem/20201027205606.jpg">

                    <div class="info">
                        <div class="title">implementation</div>
                        <div class="text">
                            According to the problems of the current new coronavirus detection platform, such as
                            cumbersome operation, large size, need to be equipped with professional operators, and
                            limited to use in the laboratory, this project is designed as a visual detection hardware
                            equipment that can be applied to community, rural hospitals and families, which is of great
                            significance in major public health safety, especially in the outbreak of epidemic.
                            <br>
                            <br>
                            Therefore, we have carried out a series of human practice research and safety assessment.
                            Through visiting community doctors, communities in high-risk areas and designated hospitals
                            to investigate their recognition and feasibility of the project, the results show that the
                            project has practical significance and is conducive to future promotion. At the same time,
                            according to the requirements of the national medical and health system, the safety of the
                            project is also considered. In the clinical validation phase of the project, virus signals
                            need to be detected, so these clinical verification work will be handed over to a special
                            laboratory for implementation. In the future plan of project implementation, we will
                            evaluate the accuracy, safety and stability of virus detection to improve the credibility of
                            the project.
                            <br>
                            <br>
                            Novel coronavirus protein markers were detected in the biological part after the importance
                            of this project was determined. The new functions of aptamer replacement system, RCA
                            isothermal amplification technology and colloidal gold strip test were used to realize the
                            three functions of protein aptamer replacement, nucleic acid signal amplification and
                            nucleic acid signal detection. For the hardware part, a microfluidic chip matching with the
                            biological part is designed, and all aspects of the detection of new coronavirus are planned
                            to be integrated into the chip. At the same time, the signal detection part plans to use
                            external equipment to build and design a fluorescence detection unit based on the mobile
                            phone platform, that is, in the dark environment, use the mobile phone flash to excite
                            fluorescence, detect the photos taken and analyze the R value of colloidal gold (Figure). In
                            terms of fluorescence detection method, inclined incidence fluorescence detection method
                            which is conducive to miniaturization is proposed. SolidWorks is used for 3D modeling, and
                            the box with microfluidic chip and mobile phone bracket matching different mobile phone
                            models are designed to achieve the goal of the initial project. In addition, due to the
                            tedious operation of pretreatment, i.e. cell lysis and other steps of the sample to be
                            tested in the user's use process, we have increased the chip chamber design to reduce the
                            impact on the product. Before the implementation, we considered the above design and thought
                            that the integration of various technologies was of great significance, and the combination
                            of centrifugal chip module and mobile phone support module was very convenient for users.
                            <br>
                            <br>
                            <img src="https://2020.igem.org/wiki/images/d/d5/T--BIT--Poster_implementation.png" alt="">
                            <div align="center" style="font-size: .8rem;">
                                Figure: original design - Mobile Phone Holder
                            </div>
                            <br>
                            <br>
                            Based on the above content, how the project can accurately complete the test and how to
                            prove the accuracy of the test is the next major consideration.
                        </div>
                    </div>

                </div>

                <div class="section third" id="contribution">
                    <img
                        src="https://cdn.jsdelivr.net/gh/a-layfolk/FigureBed/img/blog/competition/igem/20201027205606.jpg">

                    <div class="info">
                        <div class="title">contribution</div>
                        <div class="text">
                            <h2>① Problems and solutions in the process of our project design (Biology)</h2>
                            <br>
                            <br>
                            <h3>Question1：</h3>
                            <br>
                            <br>
                            In the initial stage of the project, we plan to use RPA-Isothermal amplification technology
                            to amplify the multi nucleic acid signals obtained in the sample pre-processing stage for
                            subsequent detection, including the extraction and purification of 2019-nCoV nucleic acid
                            signal (nucleic acid detection) and the complementary chain signal (protein detection) of
                            2019-nCoV characteristic protein through aptamer replacement. However, in the actual
                            experiment, we found that the length of complementary chain obtained by aptamer replacement
                            in protein detection is very short, far less than the length of primer required by RPA,
                            which shows that our project design has flaws.
                            <br>
                            <br>
                            <h3>Solution：</h3>
                            <br>
                            <br>
                            After investigation, we decided to replace RPA technology with HCR technology and RCA
                            technology. On the basis of retaining the advantages of RPA technology, such as isothermal,
                            rapid, efficient and convenient, the two technologies can greatly expand long and short
                            chain nucleic acid fragments in a short period of time, which is very consistent with the
                            needs of our project. Therefore, we initially decided to use RCA technology to amplify
                            complementary chain in protein detection and HCR technology to amplify nucleic acid signal
                            in nucleic acid detection.
                            <br>
                            <br>
                            <h3>Question 2 and 3 are omitted</h3>
                            <br>
                            <br>
                            <br>
                            <h2>② Problems and solutions during project design (hardware)</h2>
                            <br>
                            <br>
                            <h3>Question1：</h3>
                            <br>
                            <br>
                            In the early stage of our project, the design of microfluidic chip tends to use the pulling
                            whistle driven centrifugal chip based on the "pulling whistle principle". The spring rope is
                            pulled by external force to switch between the "winding" and "unfolding" states, and the
                            centrifugal disc chip is pulled to produce certain rotation speed and centrifugal force.
                            However, after our later model construction and experimental verification, the centrifugal
                            chip driven by pulling whistle has a considerable defect, that is, it cannot guarantee the
                            horizontal balance of the chip during centrifugation, and it is very unstable in the process
                            of rotation, which will lead to tremor and leakage.
                            <br>
                            <br>
                            <h3>Solution：</h3>
                            <br>
                            <br>
                            We improved the original design of pulling whistle to a microfluidic chip driven by fidget
                            spinner. The centrifugal force generated by the rotation of fidget spinner can also drive
                            the flow of liquid well. Because of the stable orbit of the shaft, it can guarantee our
                            stability in the horizontal direction.
                            <h3>Question 2，3 and 4 are omitted</h3>
                            <br>
                            <br>
                            <br>
                            <h2>③Microfluidic chip and related content production protocol</h2>
                            <br>
                            <br>
                            <h3>Synopsis of instruments：</h3>
                            <br>
                            <br>
                            The instruments used by the iGEM-BIT team in the 2020 project will be consist of our own
                            designed fingertip gyro-driven microfluidic chips, lateral flow tomographic strips,
                            temperature control modules, optical sensing modules, and intelligent external machine frame
                            structures. The goal of this project is to design a palm-held laboratory capable of
                            detecting 2019-nCoV with high accuracy and throughput.
                            In the process of the project development, we designed and completed the following
                            experiments:
                            1. Fabrication of photoresist chip
                            2. Fabrication of PMMA chip
                            3. Test strip
                            <br>
                            <br>
                            <h3>Experimental display:</h3>
                            <br>
                            <br>
                            <h3>1. Fabrication of photoresist chips:
                            </h3>
                            <h3>
                                Objective
                            </h3>
                            <br>
                            <br>
                            The basic model of the microfluidic chip based on fingertip gyroscope was fabricated using
                            the photoresist method under laboratory conditions, and the preliminary design of the chip
                            was verified.
                            <br>
                            <br>
                            <h3>Design，Experimental materials and Steps are omitted</h3>
                            <br>
                            <br>
                            <h3>2. Bonding of PMMA Chips
                            </h3>
                            <h3>
                                Objective
                            </h3>
                            The purchased processed PMMA chips were bonded together to complete the final assembly of
                            the microfluidic chips.
                            <h3>Design，Experimental materials and Steps are omitted</h3>
                            <br>
                            <br>
                            <h3>3. Preparation and assembly of colloidal gold strips</h3>
                            <h3>
                                Objective
                            </h3>
                            <br>
                            <br>
                            Preparation of colloidal gold strips for fluorescence signal detection.
                            <br>
                            <br>
                            <h3>Design，Experimental materials and Steps are omitted</h3>

                        </div>
                    </div>

                </div>

                <div class="section third" id="excellence_in_Another_Area">
                    <img src="https://2020.igem.org/wiki/images/6/62/T--BIT--Poster_Excellence_in_Another_Area.png">

                    <div class="info">
                        <div class="title">Excellence in Another Area</div>
                        <div class="text">
                            iGEMer not only needs to achieve results in the field of synthetic biology, but also needs
                            to do more work. Therefore, the BIT team hopes to expand the meaning of human practice so
                            that our work can have a more profound impact.
                            <br>
                            <br>
                            <img src="https://2020.igem.org/wiki/images/7/75/T--BIT--Poster_Excellence_in_Another_Area_1.jpg"
                                alt="">
                            <br>
                            <br>
                            <h2>
                                Science for All Alliance
                            </h2>
                            <br>
                            <br>
                            T The BIT team, SCUT-China and UESTC-Software found problems in popular science work after
                            careful research and several conference calls, and decided to join together to make changes.
                            Therefore, the three teams jointly organized and established the “Science for All Alliance”.
                            In 2020, iGEM Giant Jamboree issued a call for more iGEMers to join us!
                            The original intention of the Science for All Alliance was to help each team better conduct
                            popular science publicity activities. The Science for All Alliance hopes to form a community
                            with popular science activities as the main content, and provide a platform to provide more
                            opportunities and inspiration for all teams.
                            <br>
                            <br>
                            <h2>
                                Science for All Alliance
                            </h2>
                            <br>
                            <br>
                            Because of the impact of the COVID-19, team BIT begin to think about how to carry out
                            related work. In 2020, China will comprehensively build a well-off society. The members of
                            team BIT are also thinking about how to use their own strength to contribute to the country.
                            Therefore, team BIT got in touch with the Escort, a club of Beijing Institute of Technology.
                            They are a team of volunteers dedicated to supporting teaching activities. Team BIT got in
                            touch with them, hoping to jointly promote synthetic biology promotion lectures in the
                            future, and send knowledge and strength to more children who have no chance to contact
                            relevant knowledge.
                            <br>
                            <br>
                            <img src="https://2020.igem.org/wiki/images/e/ec/T--BIT--Poster_Excellence_in_Another_Area_2.jpg"
                                alt="">
                            <br>
                            <br>
                        </div>
                    </div>

                </div>

            </div>


            <div class="row">
                <div class="section third" id="references">
                    <img
                        src="https://cdn.jsdelivr.net/gh/a-layfolk/FigureBed/img/blog/competition/igem/20201027205606.jpg">

                    <div class="info">
                        <div class="title">References</div>
                        <div class="text">
                            [1] Zeng, W. et al. Biochemical characterization of SARS-CoV-2 nucleocapsid protein. Biochem
                            Biophys Res Commun 527, 618-623, doi:10.1016/j.bbrc.2020.04.136 (2020).
                            <br>
                            [2] Xu, X. et al. Evolution of the novel coronavirus from the ongoing Wuhan outbreak and
                            modeling of its spike protein for risk of human transmission. Sci China Life Sci 63,
                            457-460, doi:10.1007/s11427-020-1637-5 (2020).
                            <br>
                            [3] Lu, R. J. et al. Genomic characterisation and epidemiology of 2019 novel coronavirus:
                            implications for virus origins and receptor binding. Lancet 395, 565-574,
                            doi:10.1016/s0140-6736(20)30251-8 (2020).
                            <br>
                            [4] Zu, F. et al. The quenching of the fluorescence of carbon dots: A review on mechanisms
                            and applications. Microchimica Acta 184, 1899-1914, doi:10.1007/s00604-017-2318-9 (2017).
                            <br>
                            [5] Parashar, N. C., Poddar, J., Chakrabarti, S. & Parashar, G. Repurposing of SARS-CoV
                            nucleocapsid protein specific nuclease resistant RNA aptamer for therapeutics against
                            SARS-CoV-2. Infect Genet Evol 85, 104497, doi:10.1016/j.meegid.2020.104497 (2020).
                            <br>
                            [6] Journal of China Pharmaceutical University, Qiao. J. Principle of rolling circle
                            amplification and its application in medicine. 37, 201-205. (2006).
                            <br>
                            [7] Xu Jie et al. Research on label-free fluorescence detection of DNA based on DNA
                            hybridization chain reaction and hybridization gaps. 046, 1095-1101 (2018).
                            <br>
                            [8] Yang Mei, Zhang Xiaotian, Liu Xiumin, Wu Jing & China Laboratory Diagnosis, Zhu. J.
                            Application status of new coronavirus detection methods. (2020).
                            <br>
                            [9] Yang Chan et al. Application of Hybridization Chain Reaction in the Design of Biosensing
                            Probes. 37, 8-16 (2017).
                            <br>
                            [10] Bhamla, M. S. et al. Hand-powered ultralow-cost paper centrifuge. Nat. Biomed. Eng. 1,
                            0009 (2017).
                            <br>
                            [11] Michael, I. , Kim, D. , Gulenko, O. , Kumar, S. , & Cho, Y. K. . (2020). A fidget
                            spinner for the point-of-care diagnosis of urinary tract infection. Nature Biomedical
                            Engineering, 4(6), 1-10.
                            <br>
                            [12] Ren Kangning,Zhou Jianhua,Wu Hongkai. Materials for microfluidic chip fabrication.[J].
                            Accounts of chemical research,2013,46(11).
                            <br>
                            [13] Yang Yongliang. Microfluidic chip and detection device for CD4 cell counting [D].
                            Beijing University of Chemical Technology, 2016.
                            <br>
                            [14] Man Yan. Study on highly sensitive and rapid detection of protein based on microfluidic
                            chip [D]. Beijing Institute of Technology, 2015.
                            <br>
                            [15] Beijing University of Chemical Technology, 2016.
                            <br>
                            [16] Zhang Ying, Feng Hai, Wu Qiuping, Tan Zeng, he Jiabei. Research progress of colloidal
                            gold immunochromatography [J]. Advances in Modern Biomedicine, 2015 J 15 (12): 2349-2351.
                            <br>
                        </div>
                    </div>

                </div>

                <div class="section two_thirds" id="authors_and_attributions">
                    <img
                        src="https://cdn.jsdelivr.net/gh/a-layfolk/FigureBed/img/blog/competition/igem/20201027205606.jpg">

                    <div class="info">
                        <div class="title">Authors and Attributions</div>
                        <div class="text">
                            <h1>Members</h1>
                            <br>
                            <br>
                            Student Leaders: Liu Zhoumi, Mai Dina, Zhou Taiyan
                            <br>
                            <br>
                            Student Members: Guo Jiajun, Gan Daisong, Jing Ruoyan, Zhang Yibo, Kang Anshun, Wang Weilu,
                            Wang Zhiyuan, Gao Yawen, Zhang Tianjun, Zhong Hua, Gao Lianyang, Yao Xuan, Ge Xingtong, Yang
                            wenjin, Li Junlin, Wang, Zicheng
                            <br>
                            <br>
                            <img src="https://2020.igem.org/wiki/images/2/20/T--BIT--Poster_Author_1.png" alt="">
                            <div align="center" style="font-size: .8rem;">
                                Figure. members
                            </div>
                            <br>
                            <br>
                            <br>
                            <h1>Attributions</h1>
                            <br>
                            <br>
                            We'd like to thank all those who have helped us in this year, not only for making iGEM
                            possible but also for making it incredibly fun. Bless you!
                            <br>
                            <br>
                            General Support: We deeply appreciate the effort of Pro. Deng Yulin. His dedication towards
                            helping us select the ideas from the brainstorm and give us supports on the project could
                            not be more greatly appreciated. We’d love to thank Lv Xuefei, primary PI, for Supervising
                            our lab techniques as well as aiding us in project design. We also want to thank Li
                            Xiaoqiong , o ur secondary PI, for assisting us greatly with general advice. And we want to
                            thank our instructors Li Kunjie and Li Yongrui for assisting us greatly with general advice.
                            <img src="https://2020.igem.org/wiki/images/c/cc/T--BIT--Poster_Author_2.png" alt="">
                            <div align="center" style="font-size: .8rem;">
                                Figure. PIs
                            </div>
                            <br>
                            <br>
                            Bio-lab Support: We’ love to thank our student leader Medina for managing our team members
                            in charge of the biological part. We’d like to extend our thanks to Guo Jiajun, Gao
                            Lianyang, Zhang Yibo, Kang Anshun, Jing Ruoyan, for doing many things for our project. We
                            also want to thank our Wang Wenjia, Jiang Hao, Peng Zhao.
                            <br>
                            <br>
                            
                            Hardware Support: We could not be more grateful to our student leader Zhou Taiyan for
                            managing the process of hardware and chip design. We want to thank Yao Xuan, Wang Zhiyuan,
                            Gan Daisong, Zhang Tianjun, Zhonghua for the dedication in the hardware and chip design
                            part. We also want to thank He Fanglan, Wu Minjie, Li Anyi..
                            <br>
                            <br>
                            
                            Modeling Support: We want to thank Li Anyi and Zhoutai Yan for the modeling.
                            <br>
                            <br>
                            
                            Human Practices Support: We’d like to thank our student leader Liu Zhoumi for providing
                            vital connections and organizing connection events. We’d also extend our sincere thanks to
                            Wang WeiLu and Gao Yawen for their hard work.
                            <br>
                            <br>
                            
                            Wiki Support: We’d like to thank from school of computer science & technology for building
                            the wiki page. They are Li Junlin, Ge Xingtong, Wang Zicheng and Yang Wenjin.
                            <br>
                            <br>
                            
                            Special Thanks: We are very grateful to every doctor and volunteer who accepted our
                            interview. We thank BIT&GS Technology Co.Ltd for its sponsorship
                            <br>
                            <br>




                        </div>
                    </div>

                </div>
            </div>

        </div>

        <!--This is where the selected content will be displayed - Don't change this section!-->
        <div class="poster_w display">
            <h3 id="section_title"></h3>
            <p id="section_text"></p>
        </div>

        <a href="#0" class="cd-top">Back Top</a>
    </div>
</div>


<script type="text/javascript"
    src="https://2020.igem.org/wiki/index.php?title=Template:BIT/js/jquery-3.5.1.min.js&action=raw&ctype=text/javascript"></script>

<!--这是返回顶部的js代码，单独抽离之后会失效-->
<script>
    jQuery(document).ready(function ($) {
        var offset = 300,
            offset_opacity = 1200,
            scroll_top_duration = 700,
            $back_to_top = $('.cd-top');


        $(window).scroll(function () {
            ($(this).scrollTop() > offset) ? $back_to_top.addClass('cd-is-visible') : $back_to_top.removeClass('cd-is-visible cd-fade-out');
            if ($(this).scrollTop() > offset_opacity) {
                $back_to_top.addClass('cd-fade-out');
            }
        });


        $back_to_top.on('click', function (event) {
            event.preventDefault();
            $('body,html').animate({
                scrollTop: 0,
            }, scroll_top_duration
            );
        });
    });
</script>

</html>
